{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "library(MRAPSS)\n",
    "library(readr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1133.\n",
      "\n",
      "Intercept: 1.0379(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0664(0.0131).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6086.\n",
      "\n",
      "Intercept: 1.3001 (0.014).\n",
      "\n",
      "Total Observed scale h2:0.0383 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0131 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 5e-04 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "API: public: http://gwas-api.mrcieu.ac.uk/\n",
      "\n",
      "Clumping FkbBDg, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1133.\n",
      "\n",
      "Intercept: 1.0384(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.066(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1931.\n",
      "\n",
      "Intercept: 1.4611 (0.0186).\n",
      "\n",
      "Total Observed scale h2:0.0887 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0098 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -5e-04 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping A9OXkz, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0382(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0662(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.096.\n",
      "\n",
      "Intercept: 1.6301 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0578 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0188 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0039 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping scufZx, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1133.\n",
      "\n",
      "Intercept: 1.0381(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0663(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8986.\n",
      "\n",
      "Intercept: 1.2005 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0884 (0.0105).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0095 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: 0.001 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping VfwiGk, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1133.\n",
      "\n",
      "Intercept: 1.0382(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0663(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6625.\n",
      "\n",
      "Intercept: 1.8085 (0.0227).\n",
      "\n",
      "Total Observed scale h2:0.1072 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0067 (0.0082).\n",
      "\n",
      "Total Observed scale gencov: 0.0069 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping CjmB18, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0779.\n",
      "\n",
      "Intercept: 0.9717(0.0076).\n",
      "\n",
      "Total Observed scale h2:0.3197(0.0386).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6118.\n",
      "\n",
      "Intercept: 1.3009 (0.0145).\n",
      "\n",
      "Total Observed scale h2:0.0386 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0021 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: -0.0063 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yCOmQD, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.078.\n",
      "\n",
      "Intercept: 0.9716(0.0077).\n",
      "\n",
      "Total Observed scale h2:0.3203(0.0384).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1972.\n",
      "\n",
      "Intercept: 1.4639 (0.019).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0119).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0066 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0172 (0.007).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping uFSt7V, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0779.\n",
      "\n",
      "Intercept: 0.9717(0.0076).\n",
      "\n",
      "Total Observed scale h2:0.3199(0.0385).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1008.\n",
      "\n",
      "Intercept: 1.6338 (0.0169).\n",
      "\n",
      "Total Observed scale h2:0.058 (0.008).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.018 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0167 (0.0064).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qpSfw4, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0779.\n",
      "\n",
      "Intercept: 0.972(0.0074).\n",
      "\n",
      "Total Observed scale h2:0.3186(0.0378).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9031.\n",
      "\n",
      "Intercept: 1.2016 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0888 (0.0109).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0028 (0.0056).\n",
      "\n",
      "Total Observed scale gencov: -0.001 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping VUXlKO, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.078.\n",
      "\n",
      "Intercept: 0.9708(0.0074).\n",
      "\n",
      "Total Observed scale h2:0.3234(0.0377).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6698.\n",
      "\n",
      "Intercept: 1.8138 (0.0231).\n",
      "\n",
      "Total Observed scale h2:0.1073 (0.0147).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0088 (0.0089).\n",
      "\n",
      "Total Observed scale gencov: -0.0216 (0.008).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6jFknW, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4833.\n",
      "\n",
      "Intercept: 1.0327(0.0104).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6091.\n",
      "\n",
      "Intercept: 1.2955 (0.014).\n",
      "\n",
      "Total Observed scale h2:0.039 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0029 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 8e-04 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YUh2gR, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4829.\n",
      "\n",
      "Intercept: 1.0331(0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.193.\n",
      "\n",
      "Intercept: 1.453 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.0899 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0051 (0.0083).\n",
      "\n",
      "Total Observed scale gencov: -8e-04 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 7iNN7d, 2797 variants, using EUR population reference\n",
      "\n",
      "Removing 2555 of 2797 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0325(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0972.\n",
      "\n",
      "Intercept: 1.6247 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.0587 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0131 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0014 (0.0014).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PzfCWj, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4833.\n",
      "\n",
      "Intercept: 1.0332(0.0104).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9006.\n",
      "\n",
      "Intercept: 1.1988 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0107).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0029 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0011 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping H86xm1, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4828.\n",
      "\n",
      "Intercept: 1.0324(0.0104).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.66.\n",
      "\n",
      "Intercept: 1.7987 (0.0225).\n",
      "\n",
      "Total Observed scale h2:0.1083 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0143 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0012 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1wJJIh, 2797 variants, using EUR population reference\n",
      "\n",
      "Removing 2555 of 2797 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4776.\n",
      "\n",
      "Intercept: 1.9701(0.0401).\n",
      "\n",
      "Total Observed scale h2:0.4183(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6052.\n",
      "\n",
      "Intercept: 1.3049 (0.0145).\n",
      "\n",
      "Total Observed scale h2:0.0374 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.3299 (0.0114).\n",
      "\n",
      "Total Observed scale gencov: 0.0054 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping niY72C, 71718 variants, using EUR population reference\n",
      "\n",
      "Removing 69492 of 71718 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4702.\n",
      "\n",
      "Intercept: 1.9698(0.0391).\n",
      "\n",
      "Total Observed scale h2:0.4179(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1795.\n",
      "\n",
      "Intercept: 1.4644 (0.0189).\n",
      "\n",
      "Total Observed scale h2:0.0873 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.3539 (0.0126).\n",
      "\n",
      "Total Observed scale gencov: 0.0028 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping H5MmvX, 71569 variants, using EUR population reference\n",
      "\n",
      "Removing 69345 of 71569 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4774.\n",
      "\n",
      "Intercept: 1.9705(0.0397).\n",
      "\n",
      "Total Observed scale h2:0.4182(0.0175).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0932.\n",
      "\n",
      "Intercept: 1.6378 (0.0172).\n",
      "\n",
      "Total Observed scale h2:0.0566 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.4838 (0.0124).\n",
      "\n",
      "Total Observed scale gencov: -0.0199 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Vlb9d2, 71715 variants, using EUR population reference\n",
      "\n",
      "Removing 69488 of 71715 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.476.\n",
      "\n",
      "Intercept: 1.9701(0.0391).\n",
      "\n",
      "Total Observed scale h2:0.4181(0.0175).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8894.\n",
      "\n",
      "Intercept: 1.2032 (0.0181).\n",
      "\n",
      "Total Observed scale h2:0.0873 (0.0108).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.2127 (0.0106).\n",
      "\n",
      "Total Observed scale gencov: 0.0026 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1G1ZQ3, 71652 variants, using EUR population reference\n",
      "\n",
      "Removing 69427 of 71652 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4392.\n",
      "\n",
      "Intercept: 1.9717(0.0392).\n",
      "\n",
      "Total Observed scale h2:0.4167(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6434.\n",
      "\n",
      "Intercept: 1.8121 (0.0224).\n",
      "\n",
      "Total Observed scale h2:0.1061 (0.0148).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.3609 (0.0139).\n",
      "\n",
      "Total Observed scale gencov: -0.0067 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PEeFVQ, 71366 variants, using EUR population reference\n",
      "\n",
      "Removing 69139 of 71366 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.1088(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2192(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.609.\n",
      "\n",
      "Intercept: 1.2991 (0.0142).\n",
      "\n",
      "Total Observed scale h2:0.0386 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.074 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0062 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping a8izSx, 4746 variants, using EUR population reference\n",
      "\n",
      "Removing 4344 of 4746 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6914.\n",
      "\n",
      "Intercept: 1.1094(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.2188(0.0107).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1917.\n",
      "\n",
      "Intercept: 1.4599 (0.0191).\n",
      "\n",
      "Total Observed scale h2:0.089 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0265 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: 0.015 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping S3vCcK, 4745 variants, using EUR population reference\n",
      "\n",
      "Removing 4344 of 4745 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.1092(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.2189(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0975.\n",
      "\n",
      "Intercept: 1.6309 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0581 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.1374 (0.0089).\n",
      "\n",
      "Total Observed scale gencov: -0.0147 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YnFRKY, 4746 variants, using EUR population reference\n",
      "\n",
      "Removing 4344 of 4746 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6917.\n",
      "\n",
      "Intercept: 1.1089(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2191(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8984.\n",
      "\n",
      "Intercept: 1.1989 (0.0174).\n",
      "\n",
      "Total Observed scale h2:0.089 (0.0105).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.017 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0035 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GVCfaF, 4746 variants, using EUR population reference\n",
      "\n",
      "Removing 4344 of 4746 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6913.\n",
      "\n",
      "Intercept: 1.1094(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.2188(0.0108).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6612.\n",
      "\n",
      "Intercept: 1.8083 (0.0226).\n",
      "\n",
      "Total Observed scale h2:0.1075 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.1004 (0.0094).\n",
      "\n",
      "Total Observed scale gencov: -0.027 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ocPdJ0, 4745 variants, using EUR population reference\n",
      "\n",
      "Removing 4344 of 4745 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1118.\n",
      "\n",
      "Intercept: 0.9564(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1286(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6092.\n",
      "\n",
      "Intercept: 1.2975 (0.0144).\n",
      "\n",
      "Total Observed scale h2:0.0387 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0057 (0.006).\n",
      "\n",
      "Total Observed scale gencov: 0.0108 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hcmxl3, 1449 variants, using EUR population reference\n",
      "\n",
      "Removing 1314 of 1449 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1117.\n",
      "\n",
      "Intercept: 0.9568(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1282(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1924.\n",
      "\n",
      "Intercept: 1.4592 (0.0189).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0119).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0045 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0102 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 3yB2SL, 1447 variants, using EUR population reference\n",
      "\n",
      "Removing 1312 of 1447 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1117.\n",
      "\n",
      "Intercept: 0.9566(0.0098).\n",
      "\n",
      "Total Observed scale h2:0.1283(0.014).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0974.\n",
      "\n",
      "Intercept: 1.6295 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.0581 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 6e-04 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0086 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping oKFCql, 1447 variants, using EUR population reference\n",
      "\n",
      "Removing 1312 of 1447 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1118.\n",
      "\n",
      "Intercept: 0.9564(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1285(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8985.\n",
      "\n",
      "Intercept: 1.1985 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.0887 (0.0107).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0078 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0024 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XyW86z, 1447 variants, using EUR population reference\n",
      "\n",
      "Removing 1312 of 1447 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1115.\n",
      "\n",
      "Intercept: 0.9561(0.0098).\n",
      "\n",
      "Total Observed scale h2:0.1287(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.66.\n",
      "\n",
      "Intercept: 1.8053 (0.0224).\n",
      "\n",
      "Total Observed scale h2:0.1075 (0.0148).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.003 (0.0083).\n",
      "\n",
      "Total Observed scale gencov: -0.0061 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping m5sxXz, 1446 variants, using EUR population reference\n",
      "\n",
      "Removing 1311 of 1446 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.01.\n",
      "\n",
      "Intercept: 1.1126(0.0188).\n",
      "\n",
      "Total Observed scale h2:0.1849(0.0073).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6092.\n",
      "\n",
      "Intercept: 1.2997 (0.0144).\n",
      "\n",
      "Total Observed scale h2:0.0385 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.024 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: 8e-04 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping y4sZ6W, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0097.\n",
      "\n",
      "Intercept: 1.1144(0.0181).\n",
      "\n",
      "Total Observed scale h2:0.1843(0.0069).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1916.\n",
      "\n",
      "Intercept: 1.4628 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.0885 (0.0117).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0288 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0058 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping X8oGf9, 11072 variants, using EUR population reference\n",
      "\n",
      "Removing 10420 of 11072 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.01.\n",
      "\n",
      "Intercept: 1.1129(0.0189).\n",
      "\n",
      "Total Observed scale h2:0.1848(0.0072).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0964.\n",
      "\n",
      "Intercept: 1.6251 (0.0179).\n",
      "\n",
      "Total Observed scale h2:0.0585 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0297 (0.0082).\n",
      "\n",
      "Total Observed scale gencov: 0.004 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PYTmfI, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.01.\n",
      "\n",
      "Intercept: 1.1136(0.0188).\n",
      "\n",
      "Total Observed scale h2:0.1846(0.0072).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9003.\n",
      "\n",
      "Intercept: 1.2018 (0.0179).\n",
      "\n",
      "Total Observed scale h2:0.0884 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.014 (0.008).\n",
      "\n",
      "Total Observed scale gencov: 0.002 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping CtLcoA, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0067.\n",
      "\n",
      "Intercept: 1.1129(0.0193).\n",
      "\n",
      "Total Observed scale h2:0.1845(0.007).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6588.\n",
      "\n",
      "Intercept: 1.8029 (0.0227).\n",
      "\n",
      "Total Observed scale h2:0.1081 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0241 (0.0099).\n",
      "\n",
      "Total Observed scale gencov: -0.0077 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping r8KUsf, 10999 variants, using EUR population reference\n",
      "\n",
      "Removing 10347 of 10999 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6281.\n",
      "\n",
      "Intercept: 1.0193(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0694(0.0033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6096.\n",
      "\n",
      "Intercept: 1.3011 (0.0142).\n",
      "\n",
      "Total Observed scale h2:0.0383 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0124 (0.008).\n",
      "\n",
      "Total Observed scale gencov: -5e-04 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bYtJhu, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4064 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6282.\n",
      "\n",
      "Intercept: 1.0198(0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0693(0.0033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1933.\n",
      "\n",
      "Intercept: 1.4634 (0.0194).\n",
      "\n",
      "Total Observed scale h2:0.0886 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0039 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: -0.0013 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 0H3LMz, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4064 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6282.\n",
      "\n",
      "Intercept: 1.0191(0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0694(0.0033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0978.\n",
      "\n",
      "Intercept: 1.6309 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.058 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0122 (0.0087).\n",
      "\n",
      "Total Observed scale gencov: -0.001 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SemUTe, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4064 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6281.\n",
      "\n",
      "Intercept: 1.0192(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0694(0.0032).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9001.\n",
      "\n",
      "Intercept: 1.2011 (0.0174).\n",
      "\n",
      "Total Observed scale h2:0.0887 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 4e-04 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: 0.004 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping HJBSi5, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4064 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6269.\n",
      "\n",
      "Intercept: 1.0196(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0693(0.0034).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.662.\n",
      "\n",
      "Intercept: 1.8104 (0.0228).\n",
      "\n",
      "Total Observed scale h2:0.1071 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0741 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: 0.0107 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Q1KnCq, 4400 variants, using EUR population reference\n",
      "\n",
      "Removing 4058 of 4400 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BMI ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9412.\n",
      "\n",
      "Intercept: 1.1797(0.0196).\n",
      "\n",
      "Total Observed scale h2:0.2169(0.0066).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.609.\n",
      "\n",
      "Intercept: 1.2989 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.0387 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0276 (0.0087).\n",
      "\n",
      "Total Observed scale gencov: -0.0014 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SdKd5s, 28069 variants, using EUR population reference\n",
      "\n",
      "Removing 26768 of 28069 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BMI ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9409.\n",
      "\n",
      "Intercept: 1.1798(0.0202).\n",
      "\n",
      "Total Observed scale h2:0.2168(0.0066).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1917.\n",
      "\n",
      "Intercept: 1.4596 (0.0192).\n",
      "\n",
      "Total Observed scale h2:0.0891 (0.0119).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0767 (0.0109).\n",
      "\n",
      "Total Observed scale gencov: -0.0076 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping sruacp, 28044 variants, using EUR population reference\n",
      "\n",
      "Removing 26743 of 28044 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BMI ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9411.\n",
      "\n",
      "Intercept: 1.1801(0.0201).\n",
      "\n",
      "Total Observed scale h2:0.2168(0.0066).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0976.\n",
      "\n",
      "Intercept: 1.6301 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0582 (0.0081).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0365 (0.0095).\n",
      "\n",
      "Total Observed scale gencov: 0.0066 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping aAwPVw, 28069 variants, using EUR population reference\n",
      "\n",
      "Removing 26768 of 28069 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BMI ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9406.\n",
      "\n",
      "Intercept: 1.1798(0.0203).\n",
      "\n",
      "Total Observed scale h2:0.2168(0.0066).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8984.\n",
      "\n",
      "Intercept: 1.1979 (0.0179).\n",
      "\n",
      "Total Observed scale h2:0.0891 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0161 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.0019 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dujdVi, 28039 variants, using EUR population reference\n",
      "\n",
      "Removing 26739 of 28039 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "BMI ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9404.\n",
      "\n",
      "Intercept: 1.1806(0.0206).\n",
      "\n",
      "Total Observed scale h2:0.2167(0.0065).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6612.\n",
      "\n",
      "Intercept: 1.81 (0.0226).\n",
      "\n",
      "Total Observed scale h2:0.1072 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.146 (0.0117).\n",
      "\n",
      "Total Observed scale gencov: -0.0054 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nhCjSN, 28023 variants, using EUR population reference\n",
      "\n",
      "Removing 26722 of 28023 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Depression ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3826.\n",
      "\n",
      "Intercept: 1.0179(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0466(0.0029).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6099.\n",
      "\n",
      "Intercept: 1.2958 (0.0139).\n",
      "\n",
      "Total Observed scale h2:0.0391 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0172 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: 4e-04 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GPXnWI, 1376 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1376 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Depression ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3825.\n",
      "\n",
      "Intercept: 1.0182(0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0465(0.0029).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1945.\n",
      "\n",
      "Intercept: 1.4555 (0.0194).\n",
      "\n",
      "Total Observed scale h2:0.0897 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0035 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -8e-04 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pPgoZR, 1376 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1376 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Depression ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3826.\n",
      "\n",
      "Intercept: 1.0182(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0465(0.0029).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.098.\n",
      "\n",
      "Intercept: 1.6249 (0.0173).\n",
      "\n",
      "Total Observed scale h2:0.0588 (0.0079).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0224 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: 0.0015 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping csrYfV, 1376 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1376 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Depression ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3826.\n",
      "\n",
      "Intercept: 1.0177(0.0104).\n",
      "\n",
      "Total Observed scale h2:0.0466(0.0029).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9015.\n",
      "\n",
      "Intercept: 1.1991 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0025 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: -0.0015 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping KKdlpq, 1376 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1376 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Depression ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3821.\n",
      "\n",
      "Intercept: 1.0189(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0464(0.0029).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6617.\n",
      "\n",
      "Intercept: 1.8008 (0.0225).\n",
      "\n",
      "Total Observed scale h2:0.1081 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0247 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: 0.0047 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping lWVdwG, 1376 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1376 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IBD ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4925.\n",
      "\n",
      "Intercept: 1.1559(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2636(0.023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6094.\n",
      "\n",
      "Intercept: 1.2985 (0.0142).\n",
      "\n",
      "Total Observed scale h2:0.0387 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0143 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0118 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pijopg, 5087 variants, using EUR population reference\n",
      "\n",
      "Removing 4726 of 5087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IBD ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4927.\n",
      "\n",
      "Intercept: 1.157(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2628(0.0229).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1924.\n",
      "\n",
      "Intercept: 1.4604 (0.0194).\n",
      "\n",
      "Total Observed scale h2:0.089 (0.0119).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.016 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.0124 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping CeYgle, 5087 variants, using EUR population reference\n",
      "\n",
      "Removing 4726 of 5087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IBD ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4925.\n",
      "\n",
      "Intercept: 1.1556(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2639(0.0232).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0976.\n",
      "\n",
      "Intercept: 1.6335 (0.0185).\n",
      "\n",
      "Total Observed scale h2:0.0577 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0012 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: 0.0024 (0.0048).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yizNpi, 5087 variants, using EUR population reference\n",
      "\n",
      "Removing 4726 of 5087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IBD ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4925.\n",
      "\n",
      "Intercept: 1.1564(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2632(0.0234).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8998.\n",
      "\n",
      "Intercept: 1.201 (0.0184).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0105).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0018 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.003 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jZNa5R, 5087 variants, using EUR population reference\n",
      "\n",
      "Removing 4726 of 5087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IBD ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4926.\n",
      "\n",
      "Intercept: 1.1575(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.2623(0.0229).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6598.\n",
      "\n",
      "Intercept: 1.8093 (0.0223).\n",
      "\n",
      "Total Observed scale h2:0.107 (0.0148).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0356 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: -6e-04 (0.0056).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nFdIEV, 5087 variants, using EUR population reference\n",
      "\n",
      "Removing 4726 of 5087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MDD ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0672.\n",
      "\n",
      "Intercept: 0.9848(0.0072).\n",
      "\n",
      "Total Observed scale h2:0.0169(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.609.\n",
      "\n",
      "Intercept: 1.2988 (0.014).\n",
      "\n",
      "Total Observed scale h2:0.0387 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0063 (0.0061).\n",
      "\n",
      "Total Observed scale gencov: -1e-04 (0.0014).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Ao0WeV, 98 variants, using EUR population reference\n",
      "\n",
      "Removing 76 of 98 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MDD ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0671.\n",
      "\n",
      "Intercept: 0.985(0.0068).\n",
      "\n",
      "Total Observed scale h2:0.0168(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1917.\n",
      "\n",
      "Intercept: 1.4591 (0.0193).\n",
      "\n",
      "Total Observed scale h2:0.0892 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0108 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -2e-04 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MshcTV, 98 variants, using EUR population reference\n",
      "\n",
      "Removing 76 of 98 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MDD ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0672.\n",
      "\n",
      "Intercept: 0.9853(0.0071).\n",
      "\n",
      "Total Observed scale h2:0.0168(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0975.\n",
      "\n",
      "Intercept: 1.6313 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.058 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0037 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0019 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2wsq0f, 98 variants, using EUR population reference\n",
      "\n",
      "Removing 76 of 98 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MDD ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0673.\n",
      "\n",
      "Intercept: 0.9848(0.007).\n",
      "\n",
      "Total Observed scale h2:0.0169(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8984.\n",
      "\n",
      "Intercept: 1.1989 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.089 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0055 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: -0.0019 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping rMys8l, 98 variants, using EUR population reference\n",
      "\n",
      "Removing 76 of 98 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MDD ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0669.\n",
      "\n",
      "Intercept: 0.9852(0.0068).\n",
      "\n",
      "Total Observed scale h2:0.0167(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6612.\n",
      "\n",
      "Intercept: 1.808 (0.0226).\n",
      "\n",
      "Total Observed scale h2:0.1075 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0085 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0039 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yvXmGM, 98 variants, using EUR population reference\n",
      "\n",
      "Removing 76 of 98 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SCZ ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0124.\n",
      "\n",
      "Intercept: 1.0964(0.0143).\n",
      "\n",
      "Total Observed scale h2:0.4064(0.0153).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6025.\n",
      "\n",
      "Intercept: 1.302 (0.0144).\n",
      "\n",
      "Total Observed scale h2:0.0371 (0.0058).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0319 (0.0083).\n",
      "\n",
      "Total Observed scale gencov: -0.0037 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping X49WkQ, 9758 variants, using EUR population reference\n",
      "\n",
      "Removing 9093 of 9758 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SCZ ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0126.\n",
      "\n",
      "Intercept: 1.0961(0.0144).\n",
      "\n",
      "Total Observed scale h2:0.4067(0.0156).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1567.\n",
      "\n",
      "Intercept: 1.4631 (0.018).\n",
      "\n",
      "Total Observed scale h2:0.0833 (0.0107).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.022 (0.0093).\n",
      "\n",
      "Total Observed scale gencov: 0.0032 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 0f1dHf, 9758 variants, using EUR population reference\n",
      "\n",
      "Removing 9093 of 9758 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SCZ ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0124.\n",
      "\n",
      "Intercept: 1.0963(0.0142).\n",
      "\n",
      "Total Observed scale h2:0.4065(0.0152).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0778.\n",
      "\n",
      "Intercept: 1.626 (0.0164).\n",
      "\n",
      "Total Observed scale h2:0.056 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0691 (0.0105).\n",
      "\n",
      "Total Observed scale gencov: -0.0014 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping CZ6FZA, 9758 variants, using EUR population reference\n",
      "\n",
      "Removing 9093 of 9758 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SCZ ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0126.\n",
      "\n",
      "Intercept: 1.0966(0.0145).\n",
      "\n",
      "Total Observed scale h2:0.4064(0.0155).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8824.\n",
      "\n",
      "Intercept: 1.1972 (0.0173).\n",
      "\n",
      "Total Observed scale h2:0.0865 (0.0105).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0268 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0055 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping v9xE1S, 9758 variants, using EUR population reference\n",
      "\n",
      "Removing 9093 of 9758 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SCZ ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0114.\n",
      "\n",
      "Intercept: 1.0969(0.0144).\n",
      "\n",
      "Total Observed scale h2:0.4061(0.0154).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6305.\n",
      "\n",
      "Intercept: 1.8016 (0.0228).\n",
      "\n",
      "Total Observed scale h2:0.1028 (0.014).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0233 (0.0098).\n",
      "\n",
      "Total Observed scale gencov: 0.009 (0.0048).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping WN63z0, 9741 variants, using EUR population reference\n",
      "\n",
      "Removing 9076 of 9741 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Angina ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1717.\n",
      "\n",
      "Intercept: 1.0297(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0239(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.609.\n",
      "\n",
      "Intercept: 1.299 (0.014).\n",
      "\n",
      "Total Observed scale h2:0.0386 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0212 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: -0.0013 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Q15hL1, 664 variants, using EUR population reference\n",
      "\n",
      "Removing 584 of 664 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Angina ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1714.\n",
      "\n",
      "Intercept: 1.0304(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0238(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1917.\n",
      "\n",
      "Intercept: 1.4591 (0.0192).\n",
      "\n",
      "Total Observed scale h2:0.0892 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0146 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -9e-04 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping QpeI54, 664 variants, using EUR population reference\n",
      "\n",
      "Removing 584 of 664 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Angina ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1717.\n",
      "\n",
      "Intercept: 1.03(0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0239(0.0028).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0975.\n",
      "\n",
      "Intercept: 1.6314 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.058 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0344 (0.0082).\n",
      "\n",
      "Total Observed scale gencov: 0.0039 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping my3fAc, 664 variants, using EUR population reference\n",
      "\n",
      "Removing 584 of 664 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Angina ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1716.\n",
      "\n",
      "Intercept: 1.0301(0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0239(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8984.\n",
      "\n",
      "Intercept: 1.199 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0114 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: -4e-04 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LGWF71, 664 variants, using EUR population reference\n",
      "\n",
      "Removing 584 of 664 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Angina ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1702.\n",
      "\n",
      "Intercept: 1.0305(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0236(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6612.\n",
      "\n",
      "Intercept: 1.808 (0.0226).\n",
      "\n",
      "Total Observed scale h2:0.1075 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0084 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: 0.0014 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eCkdeI, 641 variants, using EUR population reference\n",
      "\n",
      "Removing 561 of 641 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CAD ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1465.\n",
      "\n",
      "Intercept: 0.8952(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.0651(0.005).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6101.\n",
      "\n",
      "Intercept: 1.3017 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.0383 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0115 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0046 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Nibdxj, 1583 variants, using EUR population reference\n",
      "\n",
      "Removing 1430 of 1583 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CAD ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1463.\n",
      "\n",
      "Intercept: 0.8957(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0649(0.005).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1935.\n",
      "\n",
      "Intercept: 1.4623 (0.0191).\n",
      "\n",
      "Total Observed scale h2:0.0887 (0.0119).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0118 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -0.0086 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping p1BX9o, 1583 variants, using EUR population reference\n",
      "\n",
      "Removing 1430 of 1583 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CAD ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1466.\n",
      "\n",
      "Intercept: 0.8954(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0651(0.005).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0987.\n",
      "\n",
      "Intercept: 1.6313 (0.0178).\n",
      "\n",
      "Total Observed scale h2:0.0581 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.028 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0035 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zjct1N, 1583 variants, using EUR population reference\n",
      "\n",
      "Removing 1430 of 1583 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CAD ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1466.\n",
      "\n",
      "Intercept: 0.895(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0652(0.0049).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8999.\n",
      "\n",
      "Intercept: 1.2012 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.0886 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0129 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: 0.0048 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LM8f4B, 1583 variants, using EUR population reference\n",
      "\n",
      "Removing 1430 of 1583 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CAD ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.146.\n",
      "\n",
      "Intercept: 0.8958(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0649(0.0049).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6634.\n",
      "\n",
      "Intercept: 1.8066 (0.0226).\n",
      "\n",
      "Total Observed scale h2:0.1078 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0075 (0.009).\n",
      "\n",
      "Total Observed scale gencov: -0.0019 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping opLzwo, 1583 variants, using EUR population reference\n",
      "\n",
      "Removing 1430 of 1583 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HBP ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0463.\n",
      "\n",
      "Intercept: 1.1249(0.0166).\n",
      "\n",
      "Total Observed scale h2:0.1137(0.0046).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.609.\n",
      "\n",
      "Intercept: 1.2979 (0.0141).\n",
      "\n",
      "Total Observed scale h2:0.0388 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0292 (0.008).\n",
      "\n",
      "Total Observed scale gencov: 0.0014 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping myETE2, 12353 variants, using EUR population reference\n",
      "\n",
      "Removing 11669 of 12353 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HBP ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0444.\n",
      "\n",
      "Intercept: 1.1255(0.017).\n",
      "\n",
      "Total Observed scale h2:0.1135(0.0046).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1917.\n",
      "\n",
      "Intercept: 1.4584 (0.0193).\n",
      "\n",
      "Total Observed scale h2:0.0893 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0619 (0.0089).\n",
      "\n",
      "Total Observed scale gencov: -0.0048 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jKGO25, 12317 variants, using EUR population reference\n",
      "\n",
      "Removing 11633 of 12317 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HBP ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.046.\n",
      "\n",
      "Intercept: 1.1256(0.0165).\n",
      "\n",
      "Total Observed scale h2:0.1135(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0975.\n",
      "\n",
      "Intercept: 1.6313 (0.0173).\n",
      "\n",
      "Total Observed scale h2:0.058 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0042 (0.0093).\n",
      "\n",
      "Total Observed scale gencov: 0 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tP0VMo, 12351 variants, using EUR population reference\n",
      "\n",
      "Removing 11667 of 12351 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HBP ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0461.\n",
      "\n",
      "Intercept: 1.1255(0.0163).\n",
      "\n",
      "Total Observed scale h2:0.1136(0.0046).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8984.\n",
      "\n",
      "Intercept: 1.1993 (0.0172).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0107).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.017 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0014 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Usi8p3, 12352 variants, using EUR population reference\n",
      "\n",
      "Removing 11668 of 12352 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HBP ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0442.\n",
      "\n",
      "Intercept: 1.1249(0.0165).\n",
      "\n",
      "Total Observed scale h2:0.1136(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6612.\n",
      "\n",
      "Intercept: 1.8093 (0.0229).\n",
      "\n",
      "Total Observed scale h2:0.1073 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.098 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: -0.0073 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2K0IEc, 12314 variants, using EUR population reference\n",
      "\n",
      "Removing 11630 of 12314 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Income ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4824.\n",
      "\n",
      "Intercept: 1.0465(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0733(0.0036).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6088.\n",
      "\n",
      "Intercept: 1.2987 (0.014).\n",
      "\n",
      "Total Observed scale h2:0.0386 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0056 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0025 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GrLXuD, 2607 variants, using EUR population reference\n",
      "\n",
      "Removing 2348 of 2607 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Income ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4818.\n",
      "\n",
      "Intercept: 1.0477(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0729(0.0036).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1915.\n",
      "\n",
      "Intercept: 1.4597 (0.0192).\n",
      "\n",
      "Total Observed scale h2:0.0891 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0097 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0079 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping X3mCcp, 2607 variants, using EUR population reference\n",
      "\n",
      "Removing 2348 of 2607 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Income ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4823.\n",
      "\n",
      "Intercept: 1.0474(0.0109).\n",
      "\n",
      "Total Observed scale h2:0.073(0.0036).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0973.\n",
      "\n",
      "Intercept: 1.6312 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.058 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0061 (0.008).\n",
      "\n",
      "Total Observed scale gencov: -0.0035 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PcY7ym, 2607 variants, using EUR population reference\n",
      "\n",
      "Removing 2348 of 2607 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Income ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4824.\n",
      "\n",
      "Intercept: 1.0468(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0732(0.0036).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8986.\n",
      "\n",
      "Intercept: 1.1994 (0.0174).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0105).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0034 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0049 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping kjQC40, 2607 variants, using EUR population reference\n",
      "\n",
      "Removing 2348 of 2607 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Income ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Mean Chi2:2.6603.\n",
      "\n",
      "Intercept: 1.8082 (0.0224).\n",
      "\n",
      "Total Observed scale h2:0.1074 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0128 (0.0098).\n",
      "\n",
      "Total Observed scale gencov: -0.0052 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9UVgAe, 2601 variants, using EUR population reference\n",
      "\n",
      "Removing 2344 of 2601 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1427.\n",
      "\n",
      "Intercept: 0.9787(0.0085).\n",
      "\n",
      "Total Observed scale h2:0.023(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6021.\n",
      "\n",
      "Intercept: 1.2964 (0.0148).\n",
      "\n",
      "Total Observed scale h2:0.0381 (0.0058).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0125 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -9e-04 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping COFsXm, 275 variants, using EUR population reference\n",
      "\n",
      "Removing 213 of 275 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1423.\n",
      "\n",
      "Intercept: 0.9784(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.023(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1708.\n",
      "\n",
      "Intercept: 1.454 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.0873 (0.0111).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0245 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: -0.001 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bGNDpH, 271 variants, using EUR population reference\n",
      "\n",
      "Removing 210 of 271 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1425.\n",
      "\n",
      "Intercept: 0.9787(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.023(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0845.\n",
      "\n",
      "Intercept: 1.6201 (0.0175).\n",
      "\n",
      "Total Observed scale h2:0.0581 (0.0079).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0016 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: -3e-04 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping y2pYuS, 272 variants, using EUR population reference\n",
      "\n",
      "Removing 210 of 272 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1425.\n",
      "\n",
      "Intercept: 0.9784(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0231(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8862.\n",
      "\n",
      "Intercept: 1.1998 (0.0183).\n",
      "\n",
      "Total Observed scale h2:0.0877 (0.0108).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0022 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0026 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XCaZYZ, 274 variants, using EUR population reference\n",
      "\n",
      "Removing 212 of 274 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1418.\n",
      "\n",
      "Intercept: 0.9797(0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0228(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6457.\n",
      "\n",
      "Intercept: 1.8018 (0.0215).\n",
      "\n",
      "Total Observed scale h2:0.1059 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.039 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: 0.0045 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qoaERd, 271 variants, using EUR population reference\n",
      "\n",
      "Removing 210 of 271 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9743(0.0098).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6101.\n",
      "\n",
      "Intercept: 1.3014 (0.0144).\n",
      "\n",
      "Total Observed scale h2:0.0384 (0.0058).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0291 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0017 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping iJHpP3, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3299.\n",
      "\n",
      "Intercept: 0.9745(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0686(0.0038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1953.\n",
      "\n",
      "Intercept: 1.4652 (0.0191).\n",
      "\n",
      "Total Observed scale h2:0.0887 (0.0119).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0155 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: -0.0018 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SQpTpy, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9744(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0968.\n",
      "\n",
      "Intercept: 1.6305 (0.0169).\n",
      "\n",
      "Total Observed scale h2:0.0582 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0481 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0019 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping J8YAfV, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3302.\n",
      "\n",
      "Intercept: 0.9742(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9027.\n",
      "\n",
      "Intercept: 1.2014 (0.0167).\n",
      "\n",
      "Total Observed scale h2:0.0892 (0.0108).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0119 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0011 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping m1kvxB, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3299.\n",
      "\n",
      "Intercept: 0.9752(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.0685(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6675.\n",
      "\n",
      "Intercept: 1.81 (0.0229).\n",
      "\n",
      "Total Observed scale h2:0.1079 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0242 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: 0.0176 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping QB7wFw, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.9046(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0949(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6016.\n",
      "\n",
      "Intercept: 1.2999 (0.0142).\n",
      "\n",
      "Total Observed scale h2:0.0377 (0.0058).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0046 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: 0.0035 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pzItb1, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1198.\n",
      "\n",
      "Intercept: 0.9054(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0944(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.171.\n",
      "\n",
      "Intercept: 1.4619 (0.0189).\n",
      "\n",
      "Total Observed scale h2:0.0863 (0.0113).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0158 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0055 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xxHAl7, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.9044(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.095(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0846.\n",
      "\n",
      "Intercept: 1.6252 (0.0178).\n",
      "\n",
      "Total Observed scale h2:0.0576 (0.008).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0168 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: 0.0046 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pdVXFe, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1201.\n",
      "\n",
      "Intercept: 0.9042(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0951(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8851.\n",
      "\n",
      "Intercept: 1.2015 (0.0185).\n",
      "\n",
      "Total Observed scale h2:0.0874 (0.011).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.015 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -5e-04 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MhSHZW, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1193.\n",
      "\n",
      "Intercept: 0.9063(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0938(0.0095).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6424.\n",
      "\n",
      "Intercept: 1.8079 (0.0222).\n",
      "\n",
      "Total Observed scale h2:0.1054 (0.0144).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0342 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: -0.0065 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eCKlGa, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0232(0.012).\n",
      "\n",
      "Total Observed scale h2:0.1824(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6295.\n",
      "\n",
      "Intercept: 1.3059 (0.0166).\n",
      "\n",
      "Total Observed scale h2:0.037 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.017 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: -0.0033 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bIKhdQ, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.318.\n",
      "\n",
      "Intercept: 1.0223(0.0118).\n",
      "\n",
      "Total Observed scale h2:0.1832(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.2488.\n",
      "\n",
      "Intercept: 1.4666 (0.021).\n",
      "\n",
      "Total Observed scale h2:0.0869 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0224 (0.009).\n",
      "\n",
      "Total Observed scale gencov: 0.0072 (0.004).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping E6idCs, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.318.\n",
      "\n",
      "Intercept: 1.0232(0.012).\n",
      "\n",
      "Total Observed scale h2:0.1825(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1266.\n",
      "\n",
      "Intercept: 1.6361 (0.0201).\n",
      "\n",
      "Total Observed scale h2:0.0561 (0.0079).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0241 (0.0103).\n",
      "\n",
      "Total Observed scale gencov: -0.0012 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LdzjuR, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0225(0.0117).\n",
      "\n",
      "Total Observed scale h2:0.183(0.0131).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9522.\n",
      "\n",
      "Intercept: 1.2026 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.0873 (0.0109).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0124 (0.0089).\n",
      "\n",
      "Total Observed scale gencov: -0.0031 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PpJvNK, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.318.\n",
      "\n",
      "Intercept: 1.0234(0.0119).\n",
      "\n",
      "Total Observed scale h2:0.1824(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.7093.\n",
      "\n",
      "Intercept: 1.8135 (0.025).\n",
      "\n",
      "Total Observed scale h2:0.1038 (0.0144).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0239 (0.0104).\n",
      "\n",
      "Total Observed scale gencov: 0.0071 (0.0051).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xZRm2b, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.122(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.3602(0.0344).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6095.\n",
      "\n",
      "Intercept: 1.2988 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.0387 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0175 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0124 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping k9Y5cl, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1228(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.3592(0.0336).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1926.\n",
      "\n",
      "Intercept: 1.4603 (0.0193).\n",
      "\n",
      "Total Observed scale h2:0.089 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0174 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.0161 (0.0059).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ilt00f, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1216(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.3608(0.0343).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0976.\n",
      "\n",
      "Intercept: 1.6328 (0.0188).\n",
      "\n",
      "Total Observed scale h2:0.0578 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0254 (0.0083).\n",
      "\n",
      "Total Observed scale gencov: 0.0085 (0.0059).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Nj4Gpx, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1226(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.3595(0.0342).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9001.\n",
      "\n",
      "Intercept: 1.2006 (0.0184).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0107).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0108 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0088 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9v34zC, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.433.\n",
      "\n",
      "Intercept: 1.1222(0.0122).\n",
      "\n",
      "Total Observed scale h2:0.36(0.0336).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6598.\n",
      "\n",
      "Intercept: 1.8091 (0.0223).\n",
      "\n",
      "Total Observed scale h2:0.1071 (0.0148).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0297 (0.0088).\n",
      "\n",
      "Total Observed scale gencov: 0.0019 (0.0065).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping TIhQQj, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0502.\n",
      "\n",
      "Intercept: 1.353(0.0254).\n",
      "\n",
      "Total Observed scale h2:0.3084(0.014).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.602.\n",
      "\n",
      "Intercept: 1.3025 (0.015).\n",
      "\n",
      "Total Observed scale h2:0.037 (0.0056).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1611 (0.01).\n",
      "\n",
      "Total Observed scale gencov: 0.0039 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eyQDzI, 26716 variants, using EUR population reference\n",
      "\n",
      "Removing 25527 of 26716 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0467.\n",
      "\n",
      "Intercept: 1.3531(0.0254).\n",
      "\n",
      "Total Observed scale h2:0.308(0.0137).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1707.\n",
      "\n",
      "Intercept: 1.4664 (0.0198).\n",
      "\n",
      "Total Observed scale h2:0.085 (0.011).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.1677 (0.0113).\n",
      "\n",
      "Total Observed scale gencov: -0.0012 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ywyCn1, 26622 variants, using EUR population reference\n",
      "\n",
      "Removing 25433 of 26622 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3533(0.0254).\n",
      "\n",
      "Total Observed scale h2:0.3083(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0851.\n",
      "\n",
      "Intercept: 1.6233 (0.0171).\n",
      "\n",
      "Total Observed scale h2:0.0573 (0.0076).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.2138 (0.0108).\n",
      "\n",
      "Total Observed scale gencov: -0.017 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1hZtbN, 26712 variants, using EUR population reference\n",
      "\n",
      "Removing 25523 of 26712 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0489.\n",
      "\n",
      "Intercept: 1.3532(0.0254).\n",
      "\n",
      "Total Observed scale h2:0.3082(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8864.\n",
      "\n",
      "Intercept: 1.2098 (0.0185).\n",
      "\n",
      "Total Observed scale h2:0.0856 (0.0109).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0859 (0.0096).\n",
      "\n",
      "Total Observed scale gencov: 0.0071 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping e2ytSs, 26682 variants, using EUR population reference\n",
      "\n",
      "Removing 25493 of 26682 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0341.\n",
      "\n",
      "Intercept: 1.3538(0.0262).\n",
      "\n",
      "Total Observed scale h2:0.3072(0.0142).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.645.\n",
      "\n",
      "Intercept: 1.8064 (0.0226).\n",
      "\n",
      "Total Observed scale h2:0.105 (0.0144).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.1721 (0.0116).\n",
      "\n",
      "Total Observed scale gencov: -0.0071 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping KGLtMz, 26501 variants, using EUR population reference\n",
      "\n",
      "Removing 25313 of 26501 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0349(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6091.\n",
      "\n",
      "Intercept: 1.2953 (0.0141).\n",
      "\n",
      "Total Observed scale h2:0.0391 (0.006).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0015 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: -7e-04 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yjj9h2, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6122.\n",
      "\n",
      "Intercept: 1.0355(0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.193.\n",
      "\n",
      "Intercept: 1.4525 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.09 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0038 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: -0.0022 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zVlFep, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.035(0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0971.\n",
      "\n",
      "Intercept: 1.6247 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.0587 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0104 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 7e-04 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping z4Gvfx, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6125.\n",
      "\n",
      "Intercept: 1.0349(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9006.\n",
      "\n",
      "Intercept: 1.1985 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.089 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0091 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0019 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Hq5n9s, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6117.\n",
      "\n",
      "Intercept: 1.0351(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6599.\n",
      "\n",
      "Intercept: 1.7985 (0.0225).\n",
      "\n",
      "Total Observed scale h2:0.1083 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0038 (0.0094).\n",
      "\n",
      "Total Observed scale gencov: 0.004 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping acWYuC, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0464(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.1086(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.609.\n",
      "\n",
      "Intercept: 1.2988 (0.014).\n",
      "\n",
      "Total Observed scale h2:0.0387 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0283 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0013 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping waQLKH, 6226 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6226 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7534.\n",
      "\n",
      "Intercept: 1.0462(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.1085(0.0046).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1917.\n",
      "\n",
      "Intercept: 1.4592 (0.0192).\n",
      "\n",
      "Total Observed scale h2:0.0892 (0.0118).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0382 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: -0.0015 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping niUKFl, 6190 variants, using EUR population reference\n",
      "\n",
      "Removing 5728 of 6190 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0464(0.013).\n",
      "\n",
      "Total Observed scale h2:0.1085(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0975.\n",
      "\n",
      "Intercept: 1.631 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.0581 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0489 (0.0093).\n",
      "\n",
      "Total Observed scale gencov: 0.0024 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 3UayXr, 6226 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6226 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7543.\n",
      "\n",
      "Intercept: 1.0462(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.1086(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8984.\n",
      "\n",
      "Intercept: 1.1995 (0.0174).\n",
      "\n",
      "Total Observed scale h2:0.0889 (0.0106).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0247 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0013 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mQ3QpO, 6226 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6226 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7524.\n",
      "\n",
      "Intercept: 1.0466(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1084(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6612.\n",
      "\n",
      "Intercept: 1.8085 (0.0224).\n",
      "\n",
      "Total Observed scale h2:0.1074 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0044 (0.01).\n",
      "\n",
      "Total Observed scale gencov: 0.004 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 98rE6W, 6179 variants, using EUR population reference\n",
      "\n",
      "Removing 5718 of 6179 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Hair_Light_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037742\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.9958(0.008).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6046.\n",
      "\n",
      "Intercept: 1.2991 (0.0147).\n",
      "\n",
      "Total Observed scale h2:0.0381 (0.0059).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0027 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1NoeH2, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Hair_Dark_Brown \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037381\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1591.\n",
      "\n",
      "Intercept: 0.9963(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.0259(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1765.\n",
      "\n",
      "Intercept: 1.4643 (0.0199).\n",
      "\n",
      "Total Observed scale h2:0.0865 (0.0112).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0185 (0.008).\n",
      "\n",
      "Total Observed scale gencov: 4e-04 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping wb5ug8, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Hair_Black \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037702\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.9962(0.0078).\n",
      "\n",
      "Total Observed scale h2:0.0259(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0874.\n",
      "\n",
      "Intercept: 1.6279 (0.0176).\n",
      "\n",
      "Total Observed scale h2:0.0574 (0.008).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Sck1wS, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Hair_Blonde \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037533\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1594.\n",
      "\n",
      "Intercept: 0.9961(0.0078).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8881.\n",
      "\n",
      "Intercept: 1.204 (0.0181).\n",
      "\n",
      "Total Observed scale h2:0.0871 (0.0108).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0103 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: -0.0013 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qX8oJw, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Tanning \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037026\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1584.\n",
      "\n",
      "Intercept: 0.9964(0.0079).\n",
      "\n",
      "Total Observed scale h2:0.0258(0.0023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.6479.\n",
      "\n",
      "Intercept: 1.8084 (0.0219).\n",
      "\n",
      "Total Observed scale h2:0.1058 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0085 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: -1e-04 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 3P4nZw, 285 variants, using EUR population reference\n",
      "\n",
      "Removing 221 of 285 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    }
   ],
   "source": [
    "## Estimating parameters for the background model using genome-wide summary statistics;\n",
    "## using plink clumping to obtain independent IVs\n",
    "\n",
    "ts1 = c(\"AD\", \"ASD\", \"Daytime_Sleepiness\", \"Height_UKB\",  \"Intelligence\", \"RA\",      \n",
    "        \"T2D\", \"Alcohol\", \"BMI\", \"Depression\", \"IBD\", \"MDD\", \"SCZ\", \"Angina\", \n",
    "        \"CAD\", \"HBP\", \"Income\", \"NEB\", \"Smoking\", \"Urate\", \"Anorexia\", \n",
    "        \"CD\", \"Height_GIANT\", \"Insomnia\", \"Neuroticism\", \"SWB\")\n",
    "\n",
    "ts2 = c(\"Hair_Light_Brown\", \"Hair_Dark_Brown\",  \"Hair_Black\",  \"Hair_Blonde\",  \"Tanning\")\n",
    "\n",
    "\n",
    "for( exposure in ts1){\n",
    "  for( outcome in ts2){\n",
    "    \n",
    "    \n",
    "    # Start the clock!\n",
    "    start = proc.time()\n",
    "    \n",
    "    # read in formatted GWAS data\n",
    "    trait1.dir = paste0(\"./GWAS_26and5_formatted/\", exposure)\n",
    "    trait2.dir = paste0(\"./GWAS_26and5_formatted/\", outcome)\n",
    "    \n",
    "    \n",
    "    cat(exposure,\"~\",outcome,\"\\n\")\n",
    "    \n",
    "    trait1 = readr::read_delim(trait1.dir, delim = \"\\t\", escape_double = FALSE, trim_ws = TRUE, progress = F)\n",
    "    \n",
    "    trait2 = readr::read_delim(trait2.dir, delim = \"\\t\", escape_double = FALSE, trim_ws = TRUE, progress = F)\n",
    "    \n",
    "    # estimate parameters for the background model\n",
    "    paras = est_paras(dat1 = trait1,\n",
    "                      dat2 = trait2,\n",
    "                      trait1.name = exposure,\n",
    "                      trait2.name = outcome,\n",
    "                      h2.fix.intercept = F,\n",
    "                      LDSC = T,\n",
    "                      ldscore.dir = \"/import/home/share/xhu/database/1KG/eur_w_ld_chr\")\n",
    "    \n",
    "    write.table(data.frame(exposure,\n",
    "                           outcome,\n",
    "                           nrow(paras$dat),\n",
    "                           rg = paras$ldsc_res$rg,\n",
    "                           rg.se = paras$ldsc_res$rg.se,\n",
    "                           paras$ldsc_res$I[1,1],\n",
    "                           paras$ldsc_res$I[2,2],\n",
    "                           rho = paras$ldsc_res$I[1,2],\n",
    "                           paras$ldsc_res$I.se[1,1],\n",
    "                           paras$ldsc_res$I.se[2,2],\n",
    "                           rho = paras$ldsc_res$I.se[1,2],\n",
    "                           paras$ldsc_res$cov[1,1],  paras$ldsc_res$cov[1,2],  paras$ldsc_res$cov[2,2],\n",
    "                           paras$ldsc_res$cov.se[1,1],  paras$ldsc_res$cov.se[1,2],  paras$ldsc_res$cov.se[2,2]),\n",
    "                file = \"pairs_ldsc_res\", col.names = F, row.names = F, quote=F,append = T)\n",
    "    \n",
    "    if(inherits(paras, 'try-error')) next\n",
    "    \n",
    "    write.table(matrix(as.vector(paras$Omega), nrow=1), paste0(\"./pairs_bg_paras/\", exposure, \"~\", outcome, \"_Omega\"),\n",
    "                row.names = F, col.names = F, quote = F)\n",
    "      \n",
    "    write.table(matrix(as.vector(paras$C), nrow=1), paste0(\"./pairs_bg_paras/\", exposure, \"~\", outcome, \"_C\"),\n",
    "                row.names = F, col.names = F, quote = F)\n",
    "    \n",
    "    # clumping      \n",
    "    cat(\"Begin clumping ...\\n \")\n",
    "    \n",
    "    clumped =  clump(paras$dat,\n",
    "                     IV.Threshold = 5e-05,\n",
    "                     SNP_col = \"SNP\",\n",
    "                     pval_col = \"pval.exp\",\n",
    "                     clump_kb = 1000,\n",
    "                     clump_r2 = 0.001,\n",
    "                     plink = \"/import/home/maxhu/plink/plink\",\n",
    "                     bfile = \"/import/home/share/xhu/database/1KG/all_1000G_EUR_Phase3\")\n",
    "    \n",
    "    if(inherits(clumped , 'try-error')) next\n",
    "    \n",
    "    \n",
    "    write.table(clumped, file = paste0(\"./MRdat/\", exposure,\"~\",outcome), col.names=T, row.names=F, quote=F)\n",
    "    \n",
    "    # Stop the clock\n",
    "    proc.time() - start\n",
    "    \n",
    "  }\n",
    "  \n",
    "}\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
