{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "library(MRAPSS)\n",
    "library(readr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 1.034(0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0704(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.075.\n",
      "\n",
      "Intercept: 0.9675 (0.007).\n",
      "\n",
      "Total Observed scale h2:0.3306 (0.0356).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -3e-04 (0.0052).\n",
      "\n",
      "Total Observed scale gencov: -0.0051 (0.0127).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "API: public: http://gwas-api.mrcieu.ac.uk/\n",
      "\n",
      "Clumping t8vIKo, 576 variants, using EUR population reference\n",
      "\n",
      "Removing 504 of 576 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1122.\n",
      "\n",
      "Intercept: 1.0358(0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0684(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4774.\n",
      "\n",
      "Intercept: 1.0326 (0.0113).\n",
      "\n",
      "Total Observed scale h2:0.048 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0011 (0.0061).\n",
      "\n",
      "Total Observed scale gencov: 3e-04 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eKGDhC, 567 variants, using EUR population reference\n",
      "\n",
      "Removing 497 of 567 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0414(0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0631(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4837.\n",
      "\n",
      "Intercept: 1.9783 (0.0417).\n",
      "\n",
      "Total Observed scale h2:0.4161 (0.0176).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0178 (0.0103).\n",
      "\n",
      "Total Observed scale gencov: -0.0208 (0.0065).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eORqBN, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0373(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.067(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6942.\n",
      "\n",
      "Intercept: 1.1117 (0.0133).\n",
      "\n",
      "Total Observed scale h2:0.2181 (0.011).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0023 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0402 (0.0069).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZfPK6C, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1136.\n",
      "\n",
      "Intercept: 1.0386(0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0661(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1118.\n",
      "\n",
      "Intercept: 0.9567 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1282 (0.0136).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0155 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: -0.008 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ldvQQO, 555 variants, using EUR population reference\n",
      "\n",
      "Removing 486 of 555 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 1.0336(0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0705(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9918.\n",
      "\n",
      "Intercept: 1.1029 (0.0183).\n",
      "\n",
      "Total Observed scale h2:0.1869 (0.0076).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0065 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0086 (0.0057).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZwNJpz, 559 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 559 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1112.\n",
      "\n",
      "Intercept: 1.0361(0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0678(0.0126).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6139.\n",
      "\n",
      "Intercept: 1.0239 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0686 (0.0032).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0 (0.0062).\n",
      "\n",
      "Total Observed scale gencov: -4e-04 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 5hiDzk, 575 variants, using EUR population reference\n",
      "\n",
      "Removing 501 of 575 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0401(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0644(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9466.\n",
      "\n",
      "Intercept: 1.1791 (0.02).\n",
      "\n",
      "Total Observed scale h2:0.2169 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0017 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: 0.0057 (0.0051).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Jlj6Xb, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1135.\n",
      "\n",
      "Intercept: 1.0369(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0674(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3836.\n",
      "\n",
      "Intercept: 1.0118 (0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0475 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0028 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0067 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping M0QPDv, 555 variants, using EUR population reference\n",
      "\n",
      "Removing 487 of 555 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1125.\n",
      "\n",
      "Intercept: 1.036(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.0675(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4921.\n",
      "\n",
      "Intercept: 1.1528 (0.013).\n",
      "\n",
      "Total Observed scale h2:0.2655 (0.0229).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0106 (0.0061).\n",
      "\n",
      "Total Observed scale gencov: 0.0039 (0.0076).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping fruU0N, 535 variants, using EUR population reference\n",
      "\n",
      "Removing 466 of 535 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0374(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0669(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0672.\n",
      "\n",
      "Intercept: 0.9862 (0.0077).\n",
      "\n",
      "Total Observed scale h2:0.0165 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.005 (0.0056).\n",
      "\n",
      "Total Observed scale gencov: -9e-04 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tYlHPG, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1121.\n",
      "\n",
      "Intercept: 1.0346(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0695(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9914.\n",
      "\n",
      "Intercept: 1.0926 (0.0141).\n",
      "\n",
      "Total Observed scale h2:0.4072 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0272 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0129 (0.007).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping y4rOle, 560 variants, using EUR population reference\n",
      "\n",
      "Removing 493 of 560 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0375(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0668(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1721.\n",
      "\n",
      "Intercept: 1.0284 (0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0242 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0083 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: -0.0024 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qqJ5RL, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1104.\n",
      "\n",
      "Intercept: 1.0337(0.0082).\n",
      "\n",
      "Total Observed scale h2:0.0699(0.0124).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1369.\n",
      "\n",
      "Intercept: 0.892 (0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0654 (0.0051).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0036 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: -1e-04 (0.004).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eNLcj8, 584 variants, using EUR population reference\n",
      "\n",
      "Removing 509 of 584 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.038(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0664(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0476.\n",
      "\n",
      "Intercept: 1.122 (0.0168).\n",
      "\n",
      "Total Observed scale h2:0.1141 (0.0047).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0102 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0061 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 7MEPpA, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1084.\n",
      "\n",
      "Intercept: 1.0417(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.0621(0.0123).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4515.\n",
      "\n",
      "Intercept: 1.0342 (0.009).\n",
      "\n",
      "Total Observed scale h2:0.0744 (0.0035).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -9e-04 (0.0053).\n",
      "\n",
      "Total Observed scale gencov: -0.0121 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dUIyRD, 595 variants, using EUR population reference\n",
      "\n",
      "Removing 516 of 595 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1085.\n",
      "\n",
      "Intercept: 1.0353(0.0073).\n",
      "\n",
      "Total Observed scale h2:0.0668(0.0122).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.138.\n",
      "\n",
      "Intercept: 0.9764 (0.0082).\n",
      "\n",
      "Total Observed scale h2:0.0234 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0111 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: -0.0019 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZMOH7D, 474 variants, using EUR population reference\n",
      "\n",
      "Removing 409 of 474 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1117.\n",
      "\n",
      "Intercept: 1.0347(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0695(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3229.\n",
      "\n",
      "Intercept: 0.9763 (0.0098).\n",
      "\n",
      "Total Observed scale h2:0.0682 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0125 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 9e-04 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping in4GVF, 552 variants, using EUR population reference\n",
      "\n",
      "Removing 480 of 552 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1056.\n",
      "\n",
      "Intercept: 1.0403(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.061(0.0122).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.108.\n",
      "\n",
      "Intercept: 0.9007 (0.0082).\n",
      "\n",
      "Total Observed scale h2:0.096 (0.009).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 7e-04 (0.0045).\n",
      "\n",
      "Total Observed scale gencov: -0.0011 (0.0047).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping K8l9cq, 500 variants, using EUR population reference\n",
      "\n",
      "Removing 425 of 500 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1167.\n",
      "\n",
      "Intercept: 1.0385(0.0118).\n",
      "\n",
      "Total Observed scale h2:0.0661(0.0142).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3069.\n",
      "\n",
      "Intercept: 1.0273 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.1784 (0.0131).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.01 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: -1e-04 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping FkNJMX, 523 variants, using EUR population reference\n",
      "\n",
      "Removing 451 of 523 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1125.\n",
      "\n",
      "Intercept: 1.0362(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.0674(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4322.\n",
      "\n",
      "Intercept: 1.1207 (0.0124).\n",
      "\n",
      "Total Observed scale h2:0.3604 (0.0324).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0072 (0.0061).\n",
      "\n",
      "Total Observed scale gencov: 1e-04 (0.0088).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping UP6OGh, 535 variants, using EUR population reference\n",
      "\n",
      "Removing 466 of 535 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1103.\n",
      "\n",
      "Intercept: 1.0364(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0667(0.0126).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9989.\n",
      "\n",
      "Intercept: 1.3444 (0.0224).\n",
      "\n",
      "Total Observed scale h2:0.3086 (0.0138).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0064 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0182 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ybSV0V, 472 variants, using EUR population reference\n",
      "\n",
      "Removing 407 of 472 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1122.\n",
      "\n",
      "Intercept: 1.0358(0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0683(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6068.\n",
      "\n",
      "Intercept: 1.0345 (0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0607 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0049 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -6e-04 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JEoCbW, 567 variants, using EUR population reference\n",
      "\n",
      "Removing 497 of 567 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.038(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0664(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7555.\n",
      "\n",
      "Intercept: 1.043 (0.0129).\n",
      "\n",
      "Total Observed scale h2:0.1092 (0.0045).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0149 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0104 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping wqTDey, 557 variants, using EUR population reference\n",
      "\n",
      "Removing 488 of 557 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AD ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1077.\n",
      "\n",
      "Intercept: 1.031(0.0076).\n",
      "\n",
      "Total Observed scale h2:0.0698(0.0123).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1554.\n",
      "\n",
      "Intercept: 0.9986 (0.0074).\n",
      "\n",
      "Total Observed scale h2:0.0254 (0.0023).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0074 (0.006).\n",
      "\n",
      "Total Observed scale gencov: -0.0043 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jWKPDN, 477 variants, using EUR population reference\n",
      "\n",
      "Removing 409 of 477 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.075.\n",
      "\n",
      "Intercept: 0.9675(0.007).\n",
      "\n",
      "Total Observed scale h2:0.3306(0.0356).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 1.034 (0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0704 (0.0135).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -3e-04 (0.0052).\n",
      "\n",
      "Total Observed scale gencov: -0.0051 (0.0127).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hC6t0S, 99 variants, using EUR population reference\n",
      "\n",
      "Removing 59 of 99 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.077.\n",
      "\n",
      "Intercept: 0.9718(0.0072).\n",
      "\n",
      "Total Observed scale h2:0.3212(0.0381).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4791.\n",
      "\n",
      "Intercept: 1.0322 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0482 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0054 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0058 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SgE5vE, 100 variants, using EUR population reference\n",
      "\n",
      "Removing 60 of 100 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0775.\n",
      "\n",
      "Intercept: 0.9689(0.0077).\n",
      "\n",
      "Total Observed scale h2:0.3283(0.0377).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.5016.\n",
      "\n",
      "Intercept: 1.982 (0.04).\n",
      "\n",
      "Total Observed scale h2:0.4182 (0.0178).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0133 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: -0.0042 (0.0111).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hQmJHn, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9721(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3184(0.038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6922.\n",
      "\n",
      "Intercept: 1.1106 (0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2176 (0.0111).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0015 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0552 (0.0135).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping BmM1jf, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9721(0.0076).\n",
      "\n",
      "Total Observed scale h2:0.3182(0.0382).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1139.\n",
      "\n",
      "Intercept: 0.956 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1303 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -9e-04 (0.0052).\n",
      "\n",
      "Total Observed scale gencov: -0.0101 (0.0122).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hPg90v, 95 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 95 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0763.\n",
      "\n",
      "Intercept: 0.9704(0.0066).\n",
      "\n",
      "Total Observed scale h2:0.3252(0.036).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9909.\n",
      "\n",
      "Intercept: 1.1009 (0.0183).\n",
      "\n",
      "Total Observed scale h2:0.1879 (0.0076).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0044 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: -0.0225 (0.0107).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8uvKIu, 100 variants, using EUR population reference\n",
      "\n",
      "Removing 59 of 100 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0748.\n",
      "\n",
      "Intercept: 0.969(0.0068).\n",
      "\n",
      "Total Observed scale h2:0.3259(0.0354).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6173.\n",
      "\n",
      "Intercept: 1.022 (0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0692 (0.0032).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0036 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0057 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZjUeAY, 101 variants, using EUR population reference\n",
      "\n",
      "Removing 60 of 101 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9716(0.0074).\n",
      "\n",
      "Total Observed scale h2:0.3199(0.0371).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9509.\n",
      "\n",
      "Intercept: 1.1815 (0.0212).\n",
      "\n",
      "Total Observed scale h2:0.2171 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0065 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0146 (0.0094).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping VSn1tV, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0789.\n",
      "\n",
      "Intercept: 0.9728(0.0078).\n",
      "\n",
      "Total Observed scale h2:0.3185(0.0392).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3835.\n",
      "\n",
      "Intercept: 1.0167 (0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0468 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0014 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0063 (0.0063).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GzvAUR, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9716(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3203(0.0372).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4946.\n",
      "\n",
      "Intercept: 1.1557 (0.013).\n",
      "\n",
      "Total Observed scale h2:0.2645 (0.024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0094 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0353 (0.0155).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zK7yyw, 94 variants, using EUR population reference\n",
      "\n",
      "Removing 56 of 94 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9719(0.0074).\n",
      "\n",
      "Total Observed scale h2:0.3192(0.0379).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0685.\n",
      "\n",
      "Intercept: 0.9866 (0.0075).\n",
      "\n",
      "Total Observed scale h2:0.0167 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0027 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: -3e-04 (0.0063).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping a6O34C, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0765.\n",
      "\n",
      "Intercept: 0.9717(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3199(0.037).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9933.\n",
      "\n",
      "Intercept: 1.0908 (0.0145).\n",
      "\n",
      "Total Observed scale h2:0.4093 (0.0155).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0134 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0727 (0.0154).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 01wqQ0, 100 variants, using EUR population reference\n",
      "\n",
      "Removing 59 of 100 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9719(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3192(0.038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1731.\n",
      "\n",
      "Intercept: 1.0305 (0.0088).\n",
      "\n",
      "Total Observed scale h2:0.024 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0023 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: -0.0069 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping g3Y0gw, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0747.\n",
      "\n",
      "Intercept: 0.9693(0.0067).\n",
      "\n",
      "Total Observed scale h2:0.3246(0.0357).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1411.\n",
      "\n",
      "Intercept: 0.8929 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0658 (0.0052).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.003 (0.0053).\n",
      "\n",
      "Total Observed scale gencov: -0.008 (0.0077).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping BzoD1U, 101 variants, using EUR population reference\n",
      "\n",
      "Removing 60 of 101 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9721(0.0077).\n",
      "\n",
      "Total Observed scale h2:0.3184(0.0388).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.051.\n",
      "\n",
      "Intercept: 1.1234 (0.0168).\n",
      "\n",
      "Total Observed scale h2:0.1143 (0.0049).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.004 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.018 (0.0078).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping uUYL1n, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0747.\n",
      "\n",
      "Intercept: 0.9693(0.0067).\n",
      "\n",
      "Total Observed scale h2:0.3249(0.0354).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4727.\n",
      "\n",
      "Intercept: 1.0407 (0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0743 (0.0037).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0125 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 2e-04 (0.0068).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2qAFZP, 101 variants, using EUR population reference\n",
      "\n",
      "Removing 60 of 101 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0764.\n",
      "\n",
      "Intercept: 0.969(0.0071).\n",
      "\n",
      "Total Observed scale h2:0.3321(0.0373).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1412.\n",
      "\n",
      "Intercept: 0.9765 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.0236 (0.002).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -1e-04 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: -0.0092 (0.0057).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ioygf1, 91 variants, using EUR population reference\n",
      "\n",
      "Removing 56 of 91 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0734.\n",
      "\n",
      "Intercept: 0.97(0.0071).\n",
      "\n",
      "Total Observed scale h2:0.3207(0.0353).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3234.\n",
      "\n",
      "Intercept: 0.9751 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0687 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0023 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: -0.0051 (0.0068).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Wkl0Jp, 95 variants, using EUR population reference\n",
      "\n",
      "Removing 56 of 95 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0743.\n",
      "\n",
      "Intercept: 0.9662(0.0074).\n",
      "\n",
      "Total Observed scale h2:0.3352(0.0347).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1188.\n",
      "\n",
      "Intercept: 0.9055 (0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0957 (0.0099).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0051 (0.0051).\n",
      "\n",
      "Total Observed scale gencov: 3e-04 (0.01).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ce9vrG, 92 variants, using EUR population reference\n",
      "\n",
      "Removing 57 of 92 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0883.\n",
      "\n",
      "Intercept: 0.9834(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.2949(0.0446).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3152.\n",
      "\n",
      "Intercept: 1.0249 (0.012).\n",
      "\n",
      "Total Observed scale h2:0.1808 (0.014).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0117 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0055 (0.0148).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping WsUd9R, 88 variants, using EUR population reference\n",
      "\n",
      "Removing 53 of 88 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9716(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3203(0.0376).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4357.\n",
      "\n",
      "Intercept: 1.12 (0.0127).\n",
      "\n",
      "Total Observed scale h2:0.3649 (0.0338).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0059 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0298 (0.019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pGZabz, 94 variants, using EUR population reference\n",
      "\n",
      "Removing 56 of 94 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0779.\n",
      "\n",
      "Intercept: 0.9688(0.008).\n",
      "\n",
      "Total Observed scale h2:0.335(0.0376).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0184.\n",
      "\n",
      "Intercept: 1.3505 (0.0237).\n",
      "\n",
      "Total Observed scale h2:0.3099 (0.0141).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0154 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0072 (0.0099).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LVTiFb, 91 variants, using EUR population reference\n",
      "\n",
      "Removing 56 of 91 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.077.\n",
      "\n",
      "Intercept: 0.9718(0.0072).\n",
      "\n",
      "Total Observed scale h2:0.3211(0.0381).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6079.\n",
      "\n",
      "Intercept: 1.0331 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.061 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0078 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -0.0043 (0.0069).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping vhDEr0, 100 variants, using EUR population reference\n",
      "\n",
      "Removing 60 of 100 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.972(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3186(0.0378).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7564.\n",
      "\n",
      "Intercept: 1.0469 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1084 (0.0045).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0064 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0207 (0.0092).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping a0KDHt, 97 variants, using EUR population reference\n",
      "\n",
      "Removing 58 of 97 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ASD ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0759.\n",
      "\n",
      "Intercept: 0.9684(0.007).\n",
      "\n",
      "Total Observed scale h2:0.3327(0.0366).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1551.\n",
      "\n",
      "Intercept: 0.9959 (0.0075).\n",
      "\n",
      "Total Observed scale h2:0.0259 (0.0023).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 7e-04 (0.0057).\n",
      "\n",
      "Total Observed scale gencov: -0.0187 (0.0066).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SoFyO6, 92 variants, using EUR population reference\n",
      "\n",
      "Removing 57 of 92 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4774.\n",
      "\n",
      "Intercept: 1.0326(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1122.\n",
      "\n",
      "Intercept: 1.0358 (0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0684 (0.0133).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0011 (0.0061).\n",
      "\n",
      "Total Observed scale gencov: 3e-04 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Ury3Ca, 2783 variants, using EUR population reference\n",
      "\n",
      "Removing 2535 of 2783 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4791.\n",
      "\n",
      "Intercept: 1.0322(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0482(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.077.\n",
      "\n",
      "Intercept: 0.9718 (0.0072).\n",
      "\n",
      "Total Observed scale h2:0.3212 (0.0381).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0054 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0058 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LRssNj, 2842 variants, using EUR population reference\n",
      "\n",
      "Removing 2594 of 2842 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4831.\n",
      "\n",
      "Intercept: 1.0356(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0476(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.494.\n",
      "\n",
      "Intercept: 1.9622 (0.0405).\n",
      "\n",
      "Total Observed scale h2:0.4211 (0.0176).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0203 (0.0106).\n",
      "\n",
      "Total Observed scale gencov: -0.0045 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ChI7Ct, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0334(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6948.\n",
      "\n",
      "Intercept: 1.1078 (0.0131).\n",
      "\n",
      "Total Observed scale h2:0.22 (0.0111).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0145 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0095 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 59AEMM, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4835.\n",
      "\n",
      "Intercept: 1.0347(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0477(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1128.\n",
      "\n",
      "Intercept: 0.9553 (0.0099).\n",
      "\n",
      "Total Observed scale h2:0.13 (0.0139).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0105 (0.006).\n",
      "\n",
      "Total Observed scale gencov: -0.002 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping TPyrJV, 2788 variants, using EUR population reference\n",
      "\n",
      "Removing 2545 of 2788 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4782.\n",
      "\n",
      "Intercept: 1.0329(0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0481(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.996.\n",
      "\n",
      "Intercept: 1.1101 (0.0182).\n",
      "\n",
      "Total Observed scale h2:0.1854 (0.0076).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0544 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.0121 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eSQi3k, 2789 variants, using EUR population reference\n",
      "\n",
      "Removing 2541 of 2789 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4774.\n",
      "\n",
      "Intercept: 1.0331(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6221.\n",
      "\n",
      "Intercept: 1.0251 (0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0689 (0.0034).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0578 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: -0.0041 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GzGDrr, 2832 variants, using EUR population reference\n",
      "\n",
      "Removing 2583 of 2832 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0351(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0476(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9528.\n",
      "\n",
      "Intercept: 1.1846 (0.0197).\n",
      "\n",
      "Total Observed scale h2:0.2169 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0745 (0.0097).\n",
      "\n",
      "Total Observed scale gencov: 0.0137 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping rBzlHy, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4837.\n",
      "\n",
      "Intercept: 1.0328(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3825.\n",
      "\n",
      "Intercept: 1.0184 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0465 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.143 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0092 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping lSKtfO, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0339(0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0478(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.495.\n",
      "\n",
      "Intercept: 1.1575 (0.0127).\n",
      "\n",
      "Total Observed scale h2:0.2634 (0.0224).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0042 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0032 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping EPFG34, 2720 variants, using EUR population reference\n",
      "\n",
      "Removing 2479 of 2720 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0331(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.068.\n",
      "\n",
      "Intercept: 0.9842 (0.0073).\n",
      "\n",
      "Total Observed scale h2:0.0171 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0479 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0035 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping oQ7FnS, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.479.\n",
      "\n",
      "Intercept: 1.0327(0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0006.\n",
      "\n",
      "Intercept: 1.0971 (0.0145).\n",
      "\n",
      "Total Observed scale h2:0.4067 (0.0151).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0044 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: 0.0132 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping TXZVSS, 2781 variants, using EUR population reference\n",
      "\n",
      "Removing 2536 of 2781 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0331(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1728.\n",
      "\n",
      "Intercept: 1.0291 (0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0242 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0354 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: 0.0048 (0.0013).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8qBOaM, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4776.\n",
      "\n",
      "Intercept: 1.0331(0.011).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.143.\n",
      "\n",
      "Intercept: 0.8947 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0652 (0.005).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0048 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 0.0086 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PF4Xdk, 2845 variants, using EUR population reference\n",
      "\n",
      "Removing 2596 of 2845 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0333(0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0529.\n",
      "\n",
      "Intercept: 1.1265 (0.0171).\n",
      "\n",
      "Total Observed scale h2:0.114 (0.0047).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0354 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0075 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping CguVdk, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4765.\n",
      "\n",
      "Intercept: 1.0324(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0481(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4782.\n",
      "\n",
      "Intercept: 1.0476 (0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0733 (0.0038).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0743 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0062 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping KyR7Ef, 2832 variants, using EUR population reference\n",
      "\n",
      "Removing 2583 of 2832 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4753.\n",
      "\n",
      "Intercept: 1.0315(0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0482(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1411.\n",
      "\n",
      "Intercept: 0.9794 (0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0231 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0033 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0023 (0.0013).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nlBggq, 2552 variants, using EUR population reference\n",
      "\n",
      "Removing 2311 of 2552 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4811.\n",
      "\n",
      "Intercept: 1.0332(0.0119).\n",
      "\n",
      "Total Observed scale h2:0.0484(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3272.\n",
      "\n",
      "Intercept: 0.9744 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0691 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0017 (0.007).\n",
      "\n",
      "Total Observed scale gencov: 0.0035 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tglvmA, 2737 variants, using EUR population reference\n",
      "\n",
      "Removing 2489 of 2737 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4729.\n",
      "\n",
      "Intercept: 1.0333(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0478(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1182.\n",
      "\n",
      "Intercept: 0.904 (0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0957 (0.0096).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0144 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yJtBk9, 2566 variants, using EUR population reference\n",
      "\n",
      "Removing 2324 of 2566 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.5179.\n",
      "\n",
      "Intercept: 1.0389(0.0141).\n",
      "\n",
      "Total Observed scale h2:0.0476(0.003).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3165.\n",
      "\n",
      "Intercept: 1.0254 (0.0124).\n",
      "\n",
      "Total Observed scale h2:0.1807 (0.0137).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0107 (0.008).\n",
      "\n",
      "Total Observed scale gencov: -0.0057 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 31SXfl, 2602 variants, using EUR population reference\n",
      "\n",
      "Removing 2381 of 2602 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4833.\n",
      "\n",
      "Intercept: 1.0336(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4348.\n",
      "\n",
      "Intercept: 1.1218 (0.012).\n",
      "\n",
      "Total Observed scale h2:0.3614 (0.033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0074 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0079 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PrRNs2, 2720 variants, using EUR population reference\n",
      "\n",
      "Removing 2479 of 2720 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4789.\n",
      "\n",
      "Intercept: 1.0331(0.011).\n",
      "\n",
      "Total Observed scale h2:0.048(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.025.\n",
      "\n",
      "Intercept: 1.356 (0.024).\n",
      "\n",
      "Total Observed scale h2:0.3082 (0.014).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0061 (0.0088).\n",
      "\n",
      "Total Observed scale gencov: -0.0029 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping C8pxdA, 2532 variants, using EUR population reference\n",
      "\n",
      "Removing 2292 of 2532 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4766.\n",
      "\n",
      "Intercept: 1.0338(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6045.\n",
      "\n",
      "Intercept: 1.0369 (0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0603 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0942 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0123 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dBnKqp, 2845 variants, using EUR population reference\n",
      "\n",
      "Removing 2596 of 2845 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0337(0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0478(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7589.\n",
      "\n",
      "Intercept: 1.046 (0.0128).\n",
      "\n",
      "Total Observed scale h2:0.109 (0.0045).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1386 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0184 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping k2ERXc, 2803 variants, using EUR population reference\n",
      "\n",
      "Removing 2560 of 2803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Daytime_Sleepiness ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4753.\n",
      "\n",
      "Intercept: 1.034(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0479(0.0027).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1578.\n",
      "\n",
      "Intercept: 0.9961 (0.0078).\n",
      "\n",
      "Total Observed scale h2:0.0261 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0079 (0.0062).\n",
      "\n",
      "Total Observed scale gencov: -0.0076 (0.0014).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qhFm7d, 2559 variants, using EUR population reference\n",
      "\n",
      "Removing 2317 of 2559 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4837.\n",
      "\n",
      "Intercept: 1.9783(0.0417).\n",
      "\n",
      "Total Observed scale h2:0.4161(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0414 (0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0631 (0.0136).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0178 (0.0103).\n",
      "\n",
      "Total Observed scale gencov: -0.0208 (0.0065).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PhAIbQ, 70299 variants, using EUR population reference\n",
      "\n",
      "Removing 68082 of 70299 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.5016.\n",
      "\n",
      "Intercept: 1.982(0.04).\n",
      "\n",
      "Total Observed scale h2:0.4182(0.0178).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0775.\n",
      "\n",
      "Intercept: 0.9689 (0.0077).\n",
      "\n",
      "Total Observed scale h2:0.3283 (0.0377).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0133 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: -0.0042 (0.0111).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ST4Mdl, 71242 variants, using EUR population reference\n",
      "\n",
      "Removing 69022 of 71242 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.494.\n",
      "\n",
      "Intercept: 1.9622(0.0405).\n",
      "\n",
      "Total Observed scale h2:0.4211(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4831.\n",
      "\n",
      "Intercept: 1.0356 (0.011).\n",
      "\n",
      "Total Observed scale h2:0.0476 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0203 (0.0106).\n",
      "\n",
      "Total Observed scale gencov: -0.0045 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8lishx, 71351 variants, using EUR population reference\n",
      "\n",
      "Removing 69125 of 71351 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.9702(0.04).\n",
      "\n",
      "Total Observed scale h2:0.4185(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6897.\n",
      "\n",
      "Intercept: 1.1102 (0.0129).\n",
      "\n",
      "Total Observed scale h2:0.218 (0.0108).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1708 (0.011).\n",
      "\n",
      "Total Observed scale gencov: 0.0472 (0.0054).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping K1xgCm, 71766 variants, using EUR population reference\n",
      "\n",
      "Removing 69539 of 71766 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4844.\n",
      "\n",
      "Intercept: 1.9754(0.0411).\n",
      "\n",
      "Total Observed scale h2:0.4171(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1113.\n",
      "\n",
      "Intercept: 0.9593 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.1256 (0.0131).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0073 (0.0087).\n",
      "\n",
      "Total Observed scale gencov: 0.0127 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GpV8NV, 71258 variants, using EUR population reference\n",
      "\n",
      "Removing 69034 of 71258 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.5101.\n",
      "\n",
      "Intercept: 1.9682(0.0417).\n",
      "\n",
      "Total Observed scale h2:0.4216(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0052.\n",
      "\n",
      "Intercept: 1.1156 (0.0184).\n",
      "\n",
      "Total Observed scale h2:0.1832 (0.0075).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0397 (0.0108).\n",
      "\n",
      "Total Observed scale gencov: -0.0174 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping FbsiKi, 69948 variants, using EUR population reference\n",
      "\n",
      "Removing 67730 of 69948 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.478.\n",
      "\n",
      "Intercept: 1.971(0.0398).\n",
      "\n",
      "Total Observed scale h2:0.4183(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9387.\n",
      "\n",
      "Intercept: 1.1795 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.2167 (0.0069).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0994 (0.0131).\n",
      "\n",
      "Total Observed scale gencov: -0.0382 (0.0053).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MDKHTo, 71762 variants, using EUR population reference\n",
      "\n",
      "Removing 69536 of 71762 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4998.\n",
      "\n",
      "Intercept: 1.9665(0.0408).\n",
      "\n",
      "Total Observed scale h2:0.4208(0.0178).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3824.\n",
      "\n",
      "Intercept: 1.0189 (0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0464 (0.0029).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0572 (0.0094).\n",
      "\n",
      "Total Observed scale gencov: -0.0106 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 5XamQn, 71359 variants, using EUR population reference\n",
      "\n",
      "Removing 69136 of 71359 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4698.\n",
      "\n",
      "Intercept: 1.9709(0.0405).\n",
      "\n",
      "Total Observed scale h2:0.4174(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4907.\n",
      "\n",
      "Intercept: 1.1616 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.257 (0.0231).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0138 (0.0098).\n",
      "\n",
      "Total Observed scale gencov: -0.0081 (0.0081).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pLVGgf, 69359 variants, using EUR population reference\n",
      "\n",
      "Removing 67135 of 69359 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.97(0.0402).\n",
      "\n",
      "Total Observed scale h2:0.4185(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0673.\n",
      "\n",
      "Intercept: 0.9839 (0.0074).\n",
      "\n",
      "Total Observed scale h2:0.0171 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0178 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: -0.0016 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping sGNTGp, 71766 variants, using EUR population reference\n",
      "\n",
      "Removing 69539 of 71766 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.5326.\n",
      "\n",
      "Intercept: 1.9656(0.0411).\n",
      "\n",
      "Total Observed scale h2:0.4213(0.0183).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.012.\n",
      "\n",
      "Intercept: 1.1004 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.4038 (0.0152).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0702 (0.0118).\n",
      "\n",
      "Total Observed scale gencov: -0.0018 (0.0067).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XuA6u5, 69739 variants, using EUR population reference\n",
      "\n",
      "Removing 67554 of 69739 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.9697(0.0401).\n",
      "\n",
      "Total Observed scale h2:0.4186(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1713.\n",
      "\n",
      "Intercept: 1.0327 (0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0233 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0698 (0.01).\n",
      "\n",
      "Total Observed scale gencov: -0.0187 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Y9fYGQ, 71766 variants, using EUR population reference\n",
      "\n",
      "Removing 69539 of 71766 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4875.\n",
      "\n",
      "Intercept: 1.9737(0.0402).\n",
      "\n",
      "Total Observed scale h2:0.418(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1458.\n",
      "\n",
      "Intercept: 0.8986 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0639 (0.0048).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0257 (0.0093).\n",
      "\n",
      "Total Observed scale gencov: -0.0214 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping RsBs47, 71619 variants, using EUR population reference\n",
      "\n",
      "Removing 69395 of 71619 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.9702(0.0397).\n",
      "\n",
      "Total Observed scale h2:0.4185(0.0178).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0442.\n",
      "\n",
      "Intercept: 1.1322 (0.0166).\n",
      "\n",
      "Total Observed scale h2:0.1122 (0.0046).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0337 (0.0116).\n",
      "\n",
      "Total Observed scale gencov: -0.0182 (0.004).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eyxH2q, 71766 variants, using EUR population reference\n",
      "\n",
      "Removing 69539 of 71766 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4809.\n",
      "\n",
      "Intercept: 1.9701(0.0399).\n",
      "\n",
      "Total Observed scale h2:0.4187(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4804.\n",
      "\n",
      "Intercept: 1.0501 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0723 (0.0037).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1134 (0.0115).\n",
      "\n",
      "Total Observed scale gencov: 0.0368 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 292U0U, 71548 variants, using EUR population reference\n",
      "\n",
      "Removing 69322 of 71548 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4114.\n",
      "\n",
      "Intercept: 1.9452(0.0396).\n",
      "\n",
      "Total Observed scale h2:0.4146(0.0172).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1427.\n",
      "\n",
      "Intercept: 0.9776 (0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0232 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0115 (0.0101).\n",
      "\n",
      "Total Observed scale gencov: -0.0033 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zFKHoY, 62787 variants, using EUR population reference\n",
      "\n",
      "Removing 60600 of 62787 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.512.\n",
      "\n",
      "Intercept: 1.9712(0.0413).\n",
      "\n",
      "Total Observed scale h2:0.4215(0.0179).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3298.\n",
      "\n",
      "Intercept: 0.9735 (0.01).\n",
      "\n",
      "Total Observed scale h2:0.0689 (0.0039).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0341 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: -0.0063 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping S6EfA6, 69109 variants, using EUR population reference\n",
      "\n",
      "Removing 66901 of 69109 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.383.\n",
      "\n",
      "Intercept: 1.9712(0.0402).\n",
      "\n",
      "Total Observed scale h2:0.4074(0.0173).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1198.\n",
      "\n",
      "Intercept: 0.9071 (0.009).\n",
      "\n",
      "Total Observed scale h2:0.0934 (0.0093).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0147 (0.0089).\n",
      "\n",
      "Total Observed scale gencov: -0.002 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9avQJZ, 63518 variants, using EUR population reference\n",
      "\n",
      "Removing 61319 of 63518 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.8141.\n",
      "\n",
      "Intercept: 2.0605(0.048).\n",
      "\n",
      "Total Observed scale h2:0.4134(0.0186).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.318.\n",
      "\n",
      "Intercept: 1.0238 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1821 (0.0145).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 4e-04 (0.0118).\n",
      "\n",
      "Total Observed scale gencov: 0.0017 (0.0063).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JTVYRj, 62735 variants, using EUR population reference\n",
      "\n",
      "Removing 60658 of 62735 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4666.\n",
      "\n",
      "Intercept: 1.9705(0.04).\n",
      "\n",
      "Total Observed scale h2:0.4175(0.0178).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4314.\n",
      "\n",
      "Intercept: 1.127 (0.012).\n",
      "\n",
      "Total Observed scale h2:0.3517 (0.0327).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0211 (0.0099).\n",
      "\n",
      "Total Observed scale gencov: -0.0072 (0.0099).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qWyq0W, 69354 variants, using EUR population reference\n",
      "\n",
      "Removing 67130 of 69354 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.444.\n",
      "\n",
      "Intercept: 1.9735(0.0409).\n",
      "\n",
      "Total Observed scale h2:0.4114(0.0175).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9801.\n",
      "\n",
      "Intercept: 1.359 (0.0206).\n",
      "\n",
      "Total Observed scale h2:0.2971 (0.0133).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.5155 (0.0255).\n",
      "\n",
      "Total Observed scale gencov: 0.3437 (0.0144).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XXGaYC, 62374 variants, using EUR population reference\n",
      "\n",
      "Removing 60198 of 62374 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4937.\n",
      "\n",
      "Intercept: 1.9617(0.0405).\n",
      "\n",
      "Total Observed scale h2:0.4212(0.0176).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6109.\n",
      "\n",
      "Intercept: 1.0375 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.06 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0425 (0.0108).\n",
      "\n",
      "Total Observed scale gencov: -0.0142 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qYUzU7, 71347 variants, using EUR population reference\n",
      "\n",
      "Removing 69121 of 71347 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.9695(0.0398).\n",
      "\n",
      "Total Observed scale h2:0.4186(0.0177).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7537.\n",
      "\n",
      "Intercept: 1.0428 (0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1093 (0.0044).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0851 (0.0118).\n",
      "\n",
      "Total Observed scale gencov: -0.012 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 4Vvest, 71766 variants, using EUR population reference\n",
      "\n",
      "Removing 69539 of 71766 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_UKB ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4055.\n",
      "\n",
      "Intercept: 1.958(0.0411).\n",
      "\n",
      "Total Observed scale h2:0.412(0.0171).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1595.\n",
      "\n",
      "Intercept: 0.9968 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.0258 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0097 (0.009).\n",
      "\n",
      "Total Observed scale gencov: 0.0025 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9xAQCU, 63004 variants, using EUR population reference\n",
      "\n",
      "Removing 60816 of 63004 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6942.\n",
      "\n",
      "Intercept: 1.1117(0.0133).\n",
      "\n",
      "Total Observed scale h2:0.2181(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.0373 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.067 (0.0134).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0023 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0402 (0.0069).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping UOVZIP, 4684 variants, using EUR population reference\n",
      "\n",
      "Removing 4282 of 4684 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6922.\n",
      "\n",
      "Intercept: 1.1106(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2176(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9721 (0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3184 (0.038).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0015 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0552 (0.0135).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tCgYAJ, 4676 variants, using EUR population reference\n",
      "\n",
      "Removing 4276 of 4676 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6948.\n",
      "\n",
      "Intercept: 1.1078(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.22(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0334 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0479 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0145 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0095 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZGofFF, 4735 variants, using EUR population reference\n",
      "\n",
      "Removing 4335 of 4735 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6897.\n",
      "\n",
      "Intercept: 1.1102(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.218(0.0108).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.9702 (0.04).\n",
      "\n",
      "Total Observed scale h2:0.4185 (0.0177).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1708 (0.011).\n",
      "\n",
      "Total Observed scale gencov: 0.0472 (0.0054).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping N3xZCq, 4713 variants, using EUR population reference\n",
      "\n",
      "Removing 4310 of 4713 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6933.\n",
      "\n",
      "Intercept: 1.11(0.0133).\n",
      "\n",
      "Total Observed scale h2:0.2187(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9571 (0.0095).\n",
      "\n",
      "Total Observed scale h2:0.1281 (0.0134).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0081 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: -0.0251 (0.0064).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping UF911t, 4725 variants, using EUR population reference\n",
      "\n",
      "Removing 4324 of 4725 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.697.\n",
      "\n",
      "Intercept: 1.1077(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2205(0.0112).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.1127 (0.0185).\n",
      "\n",
      "Total Observed scale h2:0.1848 (0.0075).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0149 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: -0.0266 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2uecvP, 4667 variants, using EUR population reference\n",
      "\n",
      "Removing 4269 of 4667 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6929.\n",
      "\n",
      "Intercept: 1.1095(0.0132).\n",
      "\n",
      "Total Observed scale h2:0.2188(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6281.\n",
      "\n",
      "Intercept: 1.0198 (0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0693 (0.0033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0307 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0088 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping HUvCk0, 4734 variants, using EUR population reference\n",
      "\n",
      "Removing 4331 of 4734 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.1107(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.2182(0.0108).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9412.\n",
      "\n",
      "Intercept: 1.1803 (0.0193).\n",
      "\n",
      "Total Observed scale h2:0.2168 (0.0069).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.046 (0.0097).\n",
      "\n",
      "Total Observed scale gencov: -0.028 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping poDiw7, 4748 variants, using EUR population reference\n",
      "\n",
      "Removing 4345 of 4748 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6953.\n",
      "\n",
      "Intercept: 1.1071(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2203(0.0112).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3826.\n",
      "\n",
      "Intercept: 1.0181 (0.01).\n",
      "\n",
      "Total Observed scale h2:0.0465 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0656 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -0.0251 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping IbqTwo, 4736 variants, using EUR population reference\n",
      "\n",
      "Removing 4336 of 4736 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6906.\n",
      "\n",
      "Intercept: 1.1075(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.2196(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4924.\n",
      "\n",
      "Intercept: 1.156 (0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2635 (0.0223).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0061 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0277 (0.0079).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MPQkEN, 4574 variants, using EUR population reference\n",
      "\n",
      "Removing 4172 of 4574 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.109(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2191(0.011).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0673.\n",
      "\n",
      "Intercept: 0.9854 (0.0073).\n",
      "\n",
      "Total Observed scale h2:0.0167 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0165 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -0.0125 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping w7xc72, 4748 variants, using EUR population reference\n",
      "\n",
      "Removing 4345 of 4748 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.698.\n",
      "\n",
      "Intercept: 1.1149(0.0135).\n",
      "\n",
      "Total Observed scale h2:0.2163(0.0112).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0124.\n",
      "\n",
      "Intercept: 1.0975 (0.0145).\n",
      "\n",
      "Total Observed scale h2:0.4058 (0.0151).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0277 (0.0087).\n",
      "\n",
      "Total Observed scale gencov: -0.0782 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping lyWnqq, 4611 variants, using EUR population reference\n",
      "\n",
      "Removing 4220 of 4611 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.109(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2191(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1717.\n",
      "\n",
      "Intercept: 1.029 (0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0241 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0246 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: -0.013 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping FvYElW, 4748 variants, using EUR population reference\n",
      "\n",
      "Removing 4345 of 4748 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6927.\n",
      "\n",
      "Intercept: 1.1092(0.0133).\n",
      "\n",
      "Total Observed scale h2:0.2189(0.0112).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1465.\n",
      "\n",
      "Intercept: 0.8953 (0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0651 (0.0049).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0124 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -0.0219 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PzfLzA, 4745 variants, using EUR population reference\n",
      "\n",
      "Removing 4342 of 4745 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.1095(0.0132).\n",
      "\n",
      "Total Observed scale h2:0.2188(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0462.\n",
      "\n",
      "Intercept: 1.1252 (0.0169).\n",
      "\n",
      "Total Observed scale h2:0.1137 (0.0046).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0233 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0181 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tSzUtn, 4748 variants, using EUR population reference\n",
      "\n",
      "Removing 4345 of 4748 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6919.\n",
      "\n",
      "Intercept: 1.1099(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2187(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4824.\n",
      "\n",
      "Intercept: 1.0484 (0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0728 (0.0037).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1206 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: 0.0728 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hEua3y, 4737 variants, using EUR population reference\n",
      "\n",
      "Removing 4334 of 4737 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6905.\n",
      "\n",
      "Intercept: 1.1097(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.2194(0.0104).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1426.\n",
      "\n",
      "Intercept: 0.979 (0.0085).\n",
      "\n",
      "Total Observed scale h2:0.023 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0011 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: -0.0123 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dPUClW, 4145 variants, using EUR population reference\n",
      "\n",
      "Removing 3760 of 4145 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6978.\n",
      "\n",
      "Intercept: 1.1093(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2206(0.0108).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9753 (0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0684 (0.0039).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0107 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: -0.0244 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping sEXdgm, 4603 variants, using EUR population reference\n",
      "\n",
      "Removing 4205 of 4603 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6846.\n",
      "\n",
      "Intercept: 1.1121(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.2171(0.0103).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.9039 (0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0953 (0.0094).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -5e-04 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0018 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping q6knCz, 4178 variants, using EUR population reference\n",
      "\n",
      "Removing 3792 of 4178 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7422.\n",
      "\n",
      "Intercept: 1.1063(0.0168).\n",
      "\n",
      "Total Observed scale h2:0.2213(0.0121).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0256 (0.0118).\n",
      "\n",
      "Total Observed scale h2:0.1807 (0.0131).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0089 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: 0.0157 (0.0076).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping enh7Ya, 4380 variants, using EUR population reference\n",
      "\n",
      "Removing 4015 of 4380 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6906.\n",
      "\n",
      "Intercept: 1.1075(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2196(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1228 (0.012).\n",
      "\n",
      "Total Observed scale h2:0.3591 (0.0318).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.013 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0305 (0.0098).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping q2B5sL, 4574 variants, using EUR population reference\n",
      "\n",
      "Removing 4172 of 4574 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.695.\n",
      "\n",
      "Intercept: 1.115(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2171(0.0102).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3525 (0.0245).\n",
      "\n",
      "Total Observed scale h2:0.3085 (0.0143).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0845 (0.009).\n",
      "\n",
      "Total Observed scale gencov: 0.0233 (0.0051).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tx3ZAr, 4119 variants, using EUR population reference\n",
      "\n",
      "Removing 3736 of 4119 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6948.\n",
      "\n",
      "Intercept: 1.1078(0.0132).\n",
      "\n",
      "Total Observed scale h2:0.2201(0.0112).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0351 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0604 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.006 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0153 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pEVyNp, 4735 variants, using EUR population reference\n",
      "\n",
      "Removing 4335 of 4735 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.1098(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2187(0.0111).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0462 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1086 (0.0043).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0408 (0.008).\n",
      "\n",
      "Total Observed scale gencov: -0.0246 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping lzbHiy, 4748 variants, using EUR population reference\n",
      "\n",
      "Removing 4345 of 4748 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Intelligence ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6874.\n",
      "\n",
      "Intercept: 1.1111(0.013).\n",
      "\n",
      "Total Observed scale h2:0.2176(0.0105).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.996 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.026 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0037 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xbAqNb, 4145 variants, using EUR population reference\n",
      "\n",
      "Removing 3761 of 4145 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1118.\n",
      "\n",
      "Intercept: 0.9567(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1282(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1136.\n",
      "\n",
      "Intercept: 1.0386 (0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0661 (0.0136).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0155 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: -0.008 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping V1UxTX, 1439 variants, using EUR population reference\n",
      "\n",
      "Removing 1305 of 1439 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1139.\n",
      "\n",
      "Intercept: 0.956(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1303(0.0142).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9721 (0.0076).\n",
      "\n",
      "Total Observed scale h2:0.3182 (0.0382).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -9e-04 (0.0052).\n",
      "\n",
      "Total Observed scale gencov: -0.0101 (0.0122).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping b7d0dZ, 1455 variants, using EUR population reference\n",
      "\n",
      "Removing 1320 of 1455 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1128.\n",
      "\n",
      "Intercept: 0.9553(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.13(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4835.\n",
      "\n",
      "Intercept: 1.0347 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0477 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0105 (0.006).\n",
      "\n",
      "Total Observed scale gencov: -0.002 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2Bu2aZ, 1455 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1455 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1113.\n",
      "\n",
      "Intercept: 0.9593(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.1256(0.0131).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4844.\n",
      "\n",
      "Intercept: 1.9754 (0.0411).\n",
      "\n",
      "Total Observed scale h2:0.4171 (0.0176).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0073 (0.0087).\n",
      "\n",
      "Total Observed scale gencov: 0.0127 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 3SqLtj, 1440 variants, using EUR population reference\n",
      "\n",
      "Removing 1306 of 1440 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9571(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.1281(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6933.\n",
      "\n",
      "Intercept: 1.11 (0.0133).\n",
      "\n",
      "Total Observed scale h2:0.2187 (0.011).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0081 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: -0.0251 (0.0064).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping baTMlG, 1456 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1456 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1139.\n",
      "\n",
      "Intercept: 0.9551(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1308(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0098.\n",
      "\n",
      "Intercept: 1.1134 (0.0185).\n",
      "\n",
      "Total Observed scale h2:0.1845 (0.0075).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0026 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0048).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping OZWMmb, 1428 variants, using EUR population reference\n",
      "\n",
      "Removing 1294 of 1428 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1124.\n",
      "\n",
      "Intercept: 0.9571(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1283(0.0137).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6279.\n",
      "\n",
      "Intercept: 1.0193 (0.0114).\n",
      "\n",
      "Total Observed scale h2:0.0693 (0.0033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0032 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: -0.0052 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MwP6Tj, 1448 variants, using EUR population reference\n",
      "\n",
      "Removing 1313 of 1448 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9585(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.1269(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9449.\n",
      "\n",
      "Intercept: 1.1791 (0.0196).\n",
      "\n",
      "Total Observed scale h2:0.2169 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0131 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0125 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping OPnJe9, 1456 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1456 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1131.\n",
      "\n",
      "Intercept: 0.9557(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.1298(0.0137).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3832.\n",
      "\n",
      "Intercept: 1.0105 (0.01).\n",
      "\n",
      "Total Observed scale h2:0.0477 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0021 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: 0.0103 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Sw0t5v, 1455 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1455 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1118.\n",
      "\n",
      "Intercept: 0.9582(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1272(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4923.\n",
      "\n",
      "Intercept: 1.1556 (0.0132).\n",
      "\n",
      "Total Observed scale h2:0.2635 (0.0228).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0268 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: -0.0258 (0.0094).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1WdyoQ, 1405 variants, using EUR population reference\n",
      "\n",
      "Removing 1271 of 1405 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9568(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.1284(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0675.\n",
      "\n",
      "Intercept: 0.9853 (0.0074).\n",
      "\n",
      "Total Observed scale h2:0.0168 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0072 (0.0053).\n",
      "\n",
      "Total Observed scale gencov: 0.0012 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xyEOGa, 1456 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1456 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1136.\n",
      "\n",
      "Intercept: 0.9553(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1297(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0136.\n",
      "\n",
      "Intercept: 1.0966 (0.0147).\n",
      "\n",
      "Total Observed scale h2:0.4066 (0.0151).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0213 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0047 (0.0076).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hj9LEU, 1388 variants, using EUR population reference\n",
      "\n",
      "Removing 1257 of 1388 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9569(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.1283(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.172.\n",
      "\n",
      "Intercept: 1.0295 (0.0088).\n",
      "\n",
      "Total Observed scale h2:0.024 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0116 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: -0.0065 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping CM6uDA, 1456 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1456 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1123.\n",
      "\n",
      "Intercept: 0.9564(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1289(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1468.\n",
      "\n",
      "Intercept: 0.8952 (0.0095).\n",
      "\n",
      "Total Observed scale h2:0.0652 (0.0049).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0055 (0.005).\n",
      "\n",
      "Total Observed scale gencov: 0.0065 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hFKd29, 1457 variants, using EUR population reference\n",
      "\n",
      "Removing 1322 of 1457 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9576(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.1277(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0463.\n",
      "\n",
      "Intercept: 1.1214 (0.0169).\n",
      "\n",
      "Total Observed scale h2:0.1142 (0.0046).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0021 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0037 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eCcTqv, 1456 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1456 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9571(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.1281(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4837.\n",
      "\n",
      "Intercept: 1.0491 (0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0728 (0.0037).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0105 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: -0.018 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping t6Z4ci, 1448 variants, using EUR population reference\n",
      "\n",
      "Removing 1313 of 1448 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1098.\n",
      "\n",
      "Intercept: 0.9556(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.128(0.0141).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1431.\n",
      "\n",
      "Intercept: 0.9791 (0.0087).\n",
      "\n",
      "Total Observed scale h2:0.023 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0021 (0.006).\n",
      "\n",
      "Total Observed scale gencov: 0.0057 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping AT9VsB, 1277 variants, using EUR population reference\n",
      "\n",
      "Removing 1147 of 1277 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1149.\n",
      "\n",
      "Intercept: 0.9561(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.1311(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3313.\n",
      "\n",
      "Intercept: 0.9737 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0689 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0023 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: 0.0199 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jORppG, 1416 variants, using EUR population reference\n",
      "\n",
      "Removing 1282 of 1416 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1073.\n",
      "\n",
      "Intercept: 0.9564(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.126(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1203.\n",
      "\n",
      "Intercept: 0.9039 (0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0953 (0.0094).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0053 (0.0056).\n",
      "\n",
      "Total Observed scale gencov: 0.0063 (0.0054).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pYHN7H, 1282 variants, using EUR population reference\n",
      "\n",
      "Removing 1150 of 1282 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1274.\n",
      "\n",
      "Intercept: 0.9638(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1262(0.0153).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3181.\n",
      "\n",
      "Intercept: 1.0246 (0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1816 (0.0135).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0014 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -2e-04 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping AO2RFz, 1342 variants, using EUR population reference\n",
      "\n",
      "Removing 1213 of 1342 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1119.\n",
      "\n",
      "Intercept: 0.9578(0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1275(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4315.\n",
      "\n",
      "Intercept: 1.1213 (0.0123).\n",
      "\n",
      "Total Observed scale h2:0.3594 (0.0329).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0188 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -0.026 (0.0107).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Hr3P5y, 1413 variants, using EUR population reference\n",
      "\n",
      "Removing 1279 of 1413 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1113.\n",
      "\n",
      "Intercept: 0.9598(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.1247(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0506.\n",
      "\n",
      "Intercept: 1.353 (0.0248).\n",
      "\n",
      "Total Observed scale h2:0.3084 (0.0143).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0025 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0076 (0.0059).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping uWPqio, 1260 variants, using EUR population reference\n",
      "\n",
      "Removing 1131 of 1260 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1128.\n",
      "\n",
      "Intercept: 0.9554(0.0098).\n",
      "\n",
      "Total Observed scale h2:0.1299(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6142.\n",
      "\n",
      "Intercept: 1.0334 (0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0607 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0017 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0099 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2wx1zf, 1455 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1455 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9569(0.0096).\n",
      "\n",
      "Total Observed scale h2:0.1283(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7556.\n",
      "\n",
      "Intercept: 1.0432 (0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1092 (0.0044).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0053 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0069 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LVevsx, 1456 variants, using EUR population reference\n",
      "\n",
      "Removing 1321 of 1456 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RA ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1093.\n",
      "\n",
      "Intercept: 0.9545(0.0102).\n",
      "\n",
      "Total Observed scale h2:0.1287(0.0144).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.159.\n",
      "\n",
      "Intercept: 0.9961 (0.0082).\n",
      "\n",
      "Total Observed scale h2:0.0259 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0037 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0026 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ndezIb, 1280 variants, using EUR population reference\n",
      "\n",
      "Removing 1149 of 1280 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9918.\n",
      "\n",
      "Intercept: 1.1029(0.0183).\n",
      "\n",
      "Total Observed scale h2:0.1869(0.0076).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 1.0336 (0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0705 (0.0134).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0065 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0086 (0.0057).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping P8JEtJ, 11048 variants, using EUR population reference\n",
      "\n",
      "Removing 10388 of 11048 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9909.\n",
      "\n",
      "Intercept: 1.1009(0.0183).\n",
      "\n",
      "Total Observed scale h2:0.1879(0.0076).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0763.\n",
      "\n",
      "Intercept: 0.9704 (0.0066).\n",
      "\n",
      "Total Observed scale h2:0.3252 (0.036).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0044 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: -0.0225 (0.0107).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6DvAxR, 11176 variants, using EUR population reference\n",
      "\n",
      "Removing 10517 of 11176 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.996.\n",
      "\n",
      "Intercept: 1.1101(0.0182).\n",
      "\n",
      "Total Observed scale h2:0.1854(0.0076).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4782.\n",
      "\n",
      "Intercept: 1.0329 (0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0481 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0544 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.0121 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping j9k2qU, 11243 variants, using EUR population reference\n",
      "\n",
      "Removing 10584 of 11243 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0052.\n",
      "\n",
      "Intercept: 1.1156(0.0184).\n",
      "\n",
      "Total Observed scale h2:0.1832(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.5101.\n",
      "\n",
      "Intercept: 1.9682 (0.0417).\n",
      "\n",
      "Total Observed scale h2:0.4216 (0.0176).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0397 (0.0108).\n",
      "\n",
      "Total Observed scale gencov: -0.0174 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping TwGe9d, 10978 variants, using EUR population reference\n",
      "\n",
      "Removing 10326 of 10978 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.1127(0.0185).\n",
      "\n",
      "Total Observed scale h2:0.1848(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.697.\n",
      "\n",
      "Intercept: 1.1077 (0.013).\n",
      "\n",
      "Total Observed scale h2:0.2205 (0.0112).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0149 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: -0.0266 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping NHiFvX, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0098.\n",
      "\n",
      "Intercept: 1.1134(0.0185).\n",
      "\n",
      "Total Observed scale h2:0.1845(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1139.\n",
      "\n",
      "Intercept: 0.9551 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.1308 (0.0139).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0026 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0048).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping oIzU2O, 10970 variants, using EUR population reference\n",
      "\n",
      "Removing 10320 of 10970 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.99.\n",
      "\n",
      "Intercept: 1.1025(0.0178).\n",
      "\n",
      "Total Observed scale h2:0.1872(0.0076).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6204.\n",
      "\n",
      "Intercept: 1.0244 (0.011).\n",
      "\n",
      "Total Observed scale h2:0.0691 (0.0033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.037 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -0.0137 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bu2JMb, 11246 variants, using EUR population reference\n",
      "\n",
      "Removing 10586 of 11246 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0089.\n",
      "\n",
      "Intercept: 1.1227(0.0181).\n",
      "\n",
      "Total Observed scale h2:0.1817(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9559.\n",
      "\n",
      "Intercept: 1.1848 (0.0201).\n",
      "\n",
      "Total Observed scale h2:0.217 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1422 (0.0116).\n",
      "\n",
      "Total Observed scale gencov: 0.1093 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nFM37S, 11066 variants, using EUR population reference\n",
      "\n",
      "Removing 10413 of 11066 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.012.\n",
      "\n",
      "Intercept: 1.1113(0.0185).\n",
      "\n",
      "Total Observed scale h2:0.1855(0.0074).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3836.\n",
      "\n",
      "Intercept: 1.0178 (0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0466 (0.0029).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0369 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.018 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping m3N68L, 11072 variants, using EUR population reference\n",
      "\n",
      "Removing 10420 of 11072 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0101.\n",
      "\n",
      "Intercept: 1.1124(0.0183).\n",
      "\n",
      "Total Observed scale h2:0.185(0.0074).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4953.\n",
      "\n",
      "Intercept: 1.1576 (0.0129).\n",
      "\n",
      "Total Observed scale h2:0.2625 (0.0227).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.032 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -9e-04 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ukjH2v, 10803 variants, using EUR population reference\n",
      "\n",
      "Removing 10155 of 10803 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.1118(0.0185).\n",
      "\n",
      "Total Observed scale h2:0.185(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0681.\n",
      "\n",
      "Intercept: 0.9846 (0.0076).\n",
      "\n",
      "Total Observed scale h2:0.017 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0168 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: 0.0088 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zOvXVd, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9997.\n",
      "\n",
      "Intercept: 1.1072(0.0192).\n",
      "\n",
      "Total Observed scale h2:0.1872(0.0077).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9969.\n",
      "\n",
      "Intercept: 1.0937 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.4085 (0.0152).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0123 (0.0083).\n",
      "\n",
      "Total Observed scale gencov: -0.0094 (0.0057).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 4tNiGd, 11175 variants, using EUR population reference\n",
      "\n",
      "Removing 10524 of 11175 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.1121(0.0184).\n",
      "\n",
      "Total Observed scale h2:0.185(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1731.\n",
      "\n",
      "Intercept: 1.0308 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0239 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0624 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: 0.0243 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Wl6fzk, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9899.\n",
      "\n",
      "Intercept: 1.1021(0.0179).\n",
      "\n",
      "Total Observed scale h2:0.1872(0.0075).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1425.\n",
      "\n",
      "Intercept: 0.8938 (0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0656 (0.005).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0324 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0436 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Y2o9uM, 11243 variants, using EUR population reference\n",
      "\n",
      "Removing 10583 of 11243 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.1121(0.0184).\n",
      "\n",
      "Total Observed scale h2:0.185(0.0073).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0528.\n",
      "\n",
      "Intercept: 1.1219 (0.0165).\n",
      "\n",
      "Total Observed scale h2:0.1145 (0.0046).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1116 (0.0093).\n",
      "\n",
      "Total Observed scale gencov: 0.062 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping C5RSbP, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9885.\n",
      "\n",
      "Intercept: 1.1007(0.0174).\n",
      "\n",
      "Total Observed scale h2:0.1876(0.0076).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.477.\n",
      "\n",
      "Intercept: 1.0452 (0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0739 (0.0039).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0468 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0239 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping lkIrd6, 11257 variants, using EUR population reference\n",
      "\n",
      "Removing 10596 of 11257 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.976.\n",
      "\n",
      "Intercept: 1.1(0.0174).\n",
      "\n",
      "Total Observed scale h2:0.1855(0.0072).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1407.\n",
      "\n",
      "Intercept: 0.9756 (0.0083).\n",
      "\n",
      "Total Observed scale h2:0.0237 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0026 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0081 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping am8vvI, 9769 variants, using EUR population reference\n",
      "\n",
      "Removing 9129 of 9769 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9872.\n",
      "\n",
      "Intercept: 1.102(0.0189).\n",
      "\n",
      "Total Observed scale h2:0.188(0.0071).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3256.\n",
      "\n",
      "Intercept: 0.9749 (0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0689 (0.0038).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0219 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: 0.02 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping NU8W8H, 10819 variants, using EUR population reference\n",
      "\n",
      "Removing 10166 of 10819 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9734.\n",
      "\n",
      "Intercept: 1.1021(0.0194).\n",
      "\n",
      "Total Observed scale h2:0.1842(0.0079).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1171.\n",
      "\n",
      "Intercept: 0.9041 (0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0954 (0.0093).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0219 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0478 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 17lkMg, 9843 variants, using EUR population reference\n",
      "\n",
      "Removing 9202 of 9843 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.085.\n",
      "\n",
      "Intercept: 1.1174(0.0209).\n",
      "\n",
      "Total Observed scale h2:0.1872(0.0083).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3167.\n",
      "\n",
      "Intercept: 1.023 (0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1823 (0.0135).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0127 (0.0095).\n",
      "\n",
      "Total Observed scale gencov: -0.0354 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Fh31G5, 10178 variants, using EUR population reference\n",
      "\n",
      "Removing 9570 of 10178 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0103.\n",
      "\n",
      "Intercept: 1.1122(0.0184).\n",
      "\n",
      "Total Observed scale h2:0.185(0.0074).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4345.\n",
      "\n",
      "Intercept: 1.1219 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.3595 (0.0337).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.026 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: -0.0103 (0.0076).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping O66Dsv, 10805 variants, using EUR population reference\n",
      "\n",
      "Removing 10157 of 10805 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9836.\n",
      "\n",
      "Intercept: 1.1053(0.0195).\n",
      "\n",
      "Total Observed scale h2:0.1843(0.0078).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.021.\n",
      "\n",
      "Intercept: 1.3513 (0.0229).\n",
      "\n",
      "Total Observed scale h2:0.3099 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.008 (0.0095).\n",
      "\n",
      "Total Observed scale gencov: -0.0114 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 4BFY4c, 9705 variants, using EUR population reference\n",
      "\n",
      "Removing 9069 of 9705 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.996.\n",
      "\n",
      "Intercept: 1.1102(0.019).\n",
      "\n",
      "Total Observed scale h2:0.1854(0.0077).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6077.\n",
      "\n",
      "Intercept: 1.0383 (0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0603 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0322 (0.0094).\n",
      "\n",
      "Total Observed scale gencov: 0.0194 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping exEaMf, 11243 variants, using EUR population reference\n",
      "\n",
      "Removing 10584 of 11243 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.112(0.0184).\n",
      "\n",
      "Total Observed scale h2:0.185(0.0074).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7608.\n",
      "\n",
      "Intercept: 1.0473 (0.013).\n",
      "\n",
      "Total Observed scale h2:0.1088 (0.0044).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0155 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: 0.0081 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping C2dIhq, 11075 variants, using EUR population reference\n",
      "\n",
      "Removing 10423 of 11075 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "T2D ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9749.\n",
      "\n",
      "Intercept: 1.1017(0.0179).\n",
      "\n",
      "Total Observed scale h2:0.1849(0.0073).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1574.\n",
      "\n",
      "Intercept: 0.9963 (0.0076).\n",
      "\n",
      "Total Observed scale h2:0.0261 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0073 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -0.0027 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 7IOCq4, 9802 variants, using EUR population reference\n",
      "\n",
      "Removing 9159 of 9802 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6139.\n",
      "\n",
      "Intercept: 1.0239(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0686(0.0032).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1112.\n",
      "\n",
      "Intercept: 1.0361 (0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0678 (0.0126).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0 (0.0062).\n",
      "\n",
      "Total Observed scale gencov: -4e-04 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8mGiOT, 4368 variants, using EUR population reference\n",
      "\n",
      "Removing 4020 of 4368 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6173.\n",
      "\n",
      "Intercept: 1.022(0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0692(0.0032).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0748.\n",
      "\n",
      "Intercept: 0.969 (0.0068).\n",
      "\n",
      "Total Observed scale h2:0.3259 (0.0354).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0036 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0057 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZOP9Mn, 4469 variants, using EUR population reference\n",
      "\n",
      "Removing 4125 of 4469 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6221.\n",
      "\n",
      "Intercept: 1.0251(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0689(0.0034).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4774.\n",
      "\n",
      "Intercept: 1.0331 (0.0107).\n",
      "\n",
      "Total Observed scale h2:0.048 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0578 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: -0.0041 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping c1LeBq, 4463 variants, using EUR population reference\n",
      "\n",
      "Removing 4118 of 4463 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6285.\n",
      "\n",
      "Intercept: 1.0195(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0694(0.0032).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4897.\n",
      "\n",
      "Intercept: 1.9745 (0.0404).\n",
      "\n",
      "Total Observed scale h2:0.4182 (0.0177).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0396 (0.0098).\n",
      "\n",
      "Total Observed scale gencov: 0.0106 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eg0Bxj, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4064 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6281.\n",
      "\n",
      "Intercept: 1.0198(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0693(0.0033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6929.\n",
      "\n",
      "Intercept: 1.1095 (0.0132).\n",
      "\n",
      "Total Observed scale h2:0.2188 (0.0111).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0307 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0088 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Y2xPAA, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4064 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6279.\n",
      "\n",
      "Intercept: 1.0193(0.0114).\n",
      "\n",
      "Total Observed scale h2:0.0693(0.0033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1124.\n",
      "\n",
      "Intercept: 0.9571 (0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1283 (0.0137).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0032 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: -0.0052 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 5wdF0Z, 4368 variants, using EUR population reference\n",
      "\n",
      "Removing 4025 of 4368 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6204.\n",
      "\n",
      "Intercept: 1.0244(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0691(0.0033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.99.\n",
      "\n",
      "Intercept: 1.1025 (0.0178).\n",
      "\n",
      "Total Observed scale h2:0.1872 (0.0076).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.037 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -0.0137 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping u59Bjx, 4388 variants, using EUR population reference\n",
      "\n",
      "Removing 4044 of 4388 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alcohol ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "IOPub message rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_msg_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_msg_rate_limit=1000.0 (msgs/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0088 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0017 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YloTlg, 282 variants, using EUR population reference\n",
      "\n",
      "Removing 216 of 282 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1539.\n",
      "\n",
      "Intercept: 0.975(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0238(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3108.\n",
      "\n",
      "Intercept: 1.0233 (0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1812 (0.0138).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -3e-04 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0037 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping kHFAXL, 262 variants, using EUR population reference\n",
      "\n",
      "Removing 202 of 262 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1426.\n",
      "\n",
      "Intercept: 0.9771(0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0233(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.427.\n",
      "\n",
      "Intercept: 1.1202 (0.0123).\n",
      "\n",
      "Total Observed scale h2:0.3567 (0.0355).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0041 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: -0.0026 (0.0047).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping U0aIyR, 266 variants, using EUR population reference\n",
      "\n",
      "Removing 205 of 266 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1399.\n",
      "\n",
      "Intercept: 0.9773(0.0079).\n",
      "\n",
      "Total Observed scale h2:0.0233(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9997.\n",
      "\n",
      "Intercept: 1.3418 (0.0225).\n",
      "\n",
      "Total Observed scale h2:0.3102 (0.0139).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0124 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0029 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Tl1fG2, 282 variants, using EUR population reference\n",
      "\n",
      "Removing 216 of 282 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1411.\n",
      "\n",
      "Intercept: 0.9794(0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0231(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6016.\n",
      "\n",
      "Intercept: 1.0358 (0.0123).\n",
      "\n",
      "Total Observed scale h2:0.0604 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0026 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0062 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JMiKNt, 282 variants, using EUR population reference\n",
      "\n",
      "Removing 216 of 282 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1426.\n",
      "\n",
      "Intercept: 0.9787(0.0085).\n",
      "\n",
      "Total Observed scale h2:0.023(0.0021).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7556.\n",
      "\n",
      "Intercept: 1.0422 (0.0129).\n",
      "\n",
      "Total Observed scale h2:0.1097 (0.0044).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0037 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0023 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping z2JvHe, 275 variants, using EUR population reference\n",
      "\n",
      "Removing 213 of 275 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NEB ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1402.\n",
      "\n",
      "Intercept: 0.9765(0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0235(0.0022).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1569.\n",
      "\n",
      "Intercept: 0.9964 (0.0074).\n",
      "\n",
      "Total Observed scale h2:0.026 (0.0023).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.016 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 0 (0.0013).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 5kHZqZ, 280 variants, using EUR population reference\n",
      "\n",
      "Removing 215 of 280 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3229.\n",
      "\n",
      "Intercept: 0.9763(0.0098).\n",
      "\n",
      "Total Observed scale h2:0.0682(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1117.\n",
      "\n",
      "Intercept: 1.0347 (0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0695 (0.0132).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0125 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 9e-04 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping C1rigD, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1227 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3234.\n",
      "\n",
      "Intercept: 0.9751(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0734.\n",
      "\n",
      "Intercept: 0.97 (0.0071).\n",
      "\n",
      "Total Observed scale h2:0.3207 (0.0353).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0023 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: -0.0051 (0.0068).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping B77zk2, 1388 variants, using EUR population reference\n",
      "\n",
      "Removing 1242 of 1388 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3272.\n",
      "\n",
      "Intercept: 0.9744(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0691(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4811.\n",
      "\n",
      "Intercept: 1.0332 (0.0119).\n",
      "\n",
      "Total Observed scale h2:0.0484 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0017 (0.007).\n",
      "\n",
      "Total Observed scale gencov: 0.0035 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping EvCNMN, 1381 variants, using EUR population reference\n",
      "\n",
      "Removing 1235 of 1381 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3298.\n",
      "\n",
      "Intercept: 0.9735(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0689(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.512.\n",
      "\n",
      "Intercept: 1.9712 (0.0413).\n",
      "\n",
      "Total Observed scale h2:0.4215 (0.0179).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0341 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: -0.0063 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tOT76o, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9753(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0684(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6978.\n",
      "\n",
      "Intercept: 1.1093 (0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2206 (0.0108).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0107 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: -0.0244 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tlBzp0, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3313.\n",
      "\n",
      "Intercept: 0.9737(0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0689(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1149.\n",
      "\n",
      "Intercept: 0.9561 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.1311 (0.0139).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0023 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: 0.0199 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping j1AGgf, 1363 variants, using EUR population reference\n",
      "\n",
      "Removing 1222 of 1363 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3256.\n",
      "\n",
      "Intercept: 0.9749(0.0088).\n",
      "\n",
      "Total Observed scale h2:0.0689(0.0038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9872.\n",
      "\n",
      "Intercept: 1.102 (0.0189).\n",
      "\n",
      "Total Observed scale h2:0.188 (0.0071).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0219 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: 0.02 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YA5Idf, 1378 variants, using EUR population reference\n",
      "\n",
      "Removing 1232 of 1378 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3211.\n",
      "\n",
      "Intercept: 0.9761(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0682(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6164.\n",
      "\n",
      "Intercept: 1.0231 (0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0691 (0.0032).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0051 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: 0.0225 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9coKK3, 1390 variants, using EUR population reference\n",
      "\n",
      "Removing 1244 of 1390 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9786(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0677(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9511.\n",
      "\n",
      "Intercept: 1.1789 (0.0204).\n",
      "\n",
      "Total Observed scale h2:0.2176 (0.0065).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0209 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0321 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping DTsQpS, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3324.\n",
      "\n",
      "Intercept: 0.9746(0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0689(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3836.\n",
      "\n",
      "Intercept: 1.0164 (0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0469 (0.0029).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0086 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0158 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping rLD4Dd, 1367 variants, using EUR population reference\n",
      "\n",
      "Removing 1224 of 1367 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3303.\n",
      "\n",
      "Intercept: 0.9762(0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0683(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4964.\n",
      "\n",
      "Intercept: 1.1562 (0.0124).\n",
      "\n",
      "Total Observed scale h2:0.2652 (0.0236).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0112 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: -0.0104 (0.0045).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jz5u6S, 1317 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1317 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9744(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0679.\n",
      "\n",
      "Intercept: 0.9858 (0.0073).\n",
      "\n",
      "Total Observed scale h2:0.0167 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0038 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 0.0094 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nah4p5, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3269.\n",
      "\n",
      "Intercept: 0.9832(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0671(0.0038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9909.\n",
      "\n",
      "Intercept: 1.0933 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.4079 (0.0157).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0138 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: 0.0224 (0.0048).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping oPro4S, 1375 variants, using EUR population reference\n",
      "\n",
      "Removing 1232 of 1375 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9744(0.0099).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1729.\n",
      "\n",
      "Intercept: 1.0315 (0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0238 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0021 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0097 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping KnIM7S, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3218.\n",
      "\n",
      "Intercept: 0.9742(0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1408.\n",
      "\n",
      "Intercept: 0.8925 (0.0096).\n",
      "\n",
      "Total Observed scale h2:0.066 (0.0051).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0069 (0.005).\n",
      "\n",
      "Total Observed scale gencov: 0.0147 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Vnknbg, 1390 variants, using EUR population reference\n",
      "\n",
      "Removing 1244 of 1390 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9738(0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0688(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0504.\n",
      "\n",
      "Intercept: 1.1209 (0.0174).\n",
      "\n",
      "Total Observed scale h2:0.1149 (0.0048).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0127 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0074 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping AvUDVI, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.321.\n",
      "\n",
      "Intercept: 0.975(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0685(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4723.\n",
      "\n",
      "Intercept: 1.0419 (0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0741 (0.0037).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 8e-04 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: -0.026 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping H3Inf7, 1390 variants, using EUR population reference\n",
      "\n",
      "Removing 1244 of 1390 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3229.\n",
      "\n",
      "Intercept: 0.9747(0.0089).\n",
      "\n",
      "Total Observed scale h2:0.069(0.0038).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1417.\n",
      "\n",
      "Intercept: 0.9748 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.024 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.007 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0095 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PVE9b6, 1230 variants, using EUR population reference\n",
      "\n",
      "Removing 1087 of 1230 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3186.\n",
      "\n",
      "Intercept: 0.9735(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0041).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1171.\n",
      "\n",
      "Intercept: 0.9036 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0959 (0.0096).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.002 (0.0056).\n",
      "\n",
      "Total Observed scale gencov: 0.0153 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping roiwA4, 1239 variants, using EUR population reference\n",
      "\n",
      "Removing 1096 of 1239 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3578.\n",
      "\n",
      "Intercept: 0.9708(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.0699(0.0042).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3173.\n",
      "\n",
      "Intercept: 1.021 (0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1848 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0038 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0036 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping efMnmK, 1278 variants, using EUR population reference\n",
      "\n",
      "Removing 1148 of 1278 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3303.\n",
      "\n",
      "Intercept: 0.9757(0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0684(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4352.\n",
      "\n",
      "Intercept: 1.1206 (0.0128).\n",
      "\n",
      "Total Observed scale h2:0.3631 (0.0341).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0151 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: -0.0204 (0.0056).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping vVI59J, 1317 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1317 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3272.\n",
      "\n",
      "Intercept: 0.9753(0.0095).\n",
      "\n",
      "Total Observed scale h2:0.0692(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0242.\n",
      "\n",
      "Intercept: 1.3516 (0.0239).\n",
      "\n",
      "Total Observed scale h2:0.311 (0.0141).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0205 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: -0.0017 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xMw56J, 1236 variants, using EUR population reference\n",
      "\n",
      "Removing 1093 of 1236 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3272.\n",
      "\n",
      "Intercept: 0.9745(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.069(0.004).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6074.\n",
      "\n",
      "Intercept: 1.0354 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0607 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0086 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0122 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YNHzF3, 1381 variants, using EUR population reference\n",
      "\n",
      "Removing 1235 of 1381 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9745(0.01).\n",
      "\n",
      "Total Observed scale h2:0.0686(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.753.\n",
      "\n",
      "Intercept: 1.0442 (0.0119).\n",
      "\n",
      "Total Observed scale h2:0.109 (0.0043).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0108 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0118 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8Vxfoh, 1370 variants, using EUR population reference\n",
      "\n",
      "Removing 1228 of 1370 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Smoking ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3221.\n",
      "\n",
      "Intercept: 0.9754(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0687(0.0039).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1557.\n",
      "\n",
      "Intercept: 0.9958 (0.0076).\n",
      "\n",
      "Total Observed scale h2:0.026 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0167 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: -0.0086 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MVrOGr, 1229 variants, using EUR population reference\n",
      "\n",
      "Removing 1087 of 1229 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.108.\n",
      "\n",
      "Intercept: 0.9007(0.0082).\n",
      "\n",
      "Total Observed scale h2:0.096(0.009).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1056.\n",
      "\n",
      "Intercept: 1.0403 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.061 (0.0122).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 7e-04 (0.0045).\n",
      "\n",
      "Total Observed scale gencov: -0.0011 (0.0047).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bQgEui, 1283 variants, using EUR population reference\n",
      "\n",
      "Removing 1184 of 1283 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1188.\n",
      "\n",
      "Intercept: 0.9055(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0957(0.0099).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0743.\n",
      "\n",
      "Intercept: 0.9662 (0.0074).\n",
      "\n",
      "Total Observed scale h2:0.3352 (0.0347).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0051 (0.0051).\n",
      "\n",
      "Total Observed scale gencov: 3e-04 (0.01).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping DEVajk, 1281 variants, using EUR population reference\n",
      "\n",
      "Removing 1184 of 1281 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1182.\n",
      "\n",
      "Intercept: 0.904(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0957(0.0096).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4729.\n",
      "\n",
      "Intercept: 1.0333 (0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0478 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0144 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8P7cfN, 1247 variants, using EUR population reference\n",
      "\n",
      "Removing 1150 of 1247 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1198.\n",
      "\n",
      "Intercept: 0.9071(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0934(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.383.\n",
      "\n",
      "Intercept: 1.9712 (0.0402).\n",
      "\n",
      "Total Observed scale h2:0.4074 (0.0173).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0147 (0.0089).\n",
      "\n",
      "Total Observed scale gencov: -0.002 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6siXrl, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.9039(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0953(0.0094).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6846.\n",
      "\n",
      "Intercept: 1.1121 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.2171 (0.0103).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -5e-04 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0018 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bR0OdT, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1203.\n",
      "\n",
      "Intercept: 0.9039(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0953(0.0094).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1073.\n",
      "\n",
      "Intercept: 0.9564 (0.0096).\n",
      "\n",
      "Total Observed scale h2:0.126 (0.0133).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0053 (0.0056).\n",
      "\n",
      "Total Observed scale gencov: 0.0063 (0.0054).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping BXjnGd, 1222 variants, using EUR population reference\n",
      "\n",
      "Removing 1129 of 1222 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1171.\n",
      "\n",
      "Intercept: 0.9041(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0954(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9734.\n",
      "\n",
      "Intercept: 1.1021 (0.0194).\n",
      "\n",
      "Total Observed scale h2:0.1842 (0.0079).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0219 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0478 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6CLbE1, 1201 variants, using EUR population reference\n",
      "\n",
      "Removing 1107 of 1201 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1166.\n",
      "\n",
      "Intercept: 0.905(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0951(0.0095).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6042.\n",
      "\n",
      "Intercept: 1.0194 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0688 (0.0031).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0095 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 6e-04 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ssyADw, 1275 variants, using EUR population reference\n",
      "\n",
      "Removing 1179 of 1275 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1199.\n",
      "\n",
      "Intercept: 0.9057(0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0942(0.0092).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9114.\n",
      "\n",
      "Intercept: 1.1804 (0.0214).\n",
      "\n",
      "Total Observed scale h2:0.2145 (0.0067).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.022 (0.008).\n",
      "\n",
      "Total Observed scale gencov: 0.05 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JxlkKi, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1218.\n",
      "\n",
      "Intercept: 0.9054(0.0093).\n",
      "\n",
      "Total Observed scale h2:0.095(0.0094).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3805.\n",
      "\n",
      "Intercept: 1.0081 (0.0104).\n",
      "\n",
      "Total Observed scale h2:0.0481 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0035 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: 0.0055 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 8Fiov9, 1221 variants, using EUR population reference\n",
      "\n",
      "Removing 1128 of 1221 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.119.\n",
      "\n",
      "Intercept: 0.9056(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0941(0.0094).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.479.\n",
      "\n",
      "Intercept: 1.1571 (0.0123).\n",
      "\n",
      "Total Observed scale h2:0.254 (0.023).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0038 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: 0.0039 (0.007).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping LWY2FP, 1168 variants, using EUR population reference\n",
      "\n",
      "Removing 1075 of 1168 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.904(0.0092).\n",
      "\n",
      "Total Observed scale h2:0.0952(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0673.\n",
      "\n",
      "Intercept: 0.9841 (0.0074).\n",
      "\n",
      "Total Observed scale h2:0.0171 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0058 (0.0051).\n",
      "\n",
      "Total Observed scale gencov: 0.0058 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping U9HAuq, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1193.\n",
      "\n",
      "Intercept: 0.9052(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0956(0.0095).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9758.\n",
      "\n",
      "Intercept: 1.0952 (0.0137).\n",
      "\n",
      "Total Observed scale h2:0.4028 (0.0149).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0221 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0269 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2ULGhi, 1255 variants, using EUR population reference\n",
      "\n",
      "Removing 1160 of 1255 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.904(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0952(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1696.\n",
      "\n",
      "Intercept: 1.0323 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0233 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0054 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 0.0103 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping neeH4w, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1145.\n",
      "\n",
      "Intercept: 0.9025(0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0962(0.0092).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.129.\n",
      "\n",
      "Intercept: 0.8885 (0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0649 (0.0049).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0098 (0.0055).\n",
      "\n",
      "Total Observed scale gencov: 0.0148 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping kuAo1a, 1302 variants, using EUR population reference\n",
      "\n",
      "Removing 1204 of 1302 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.904(0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0952(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0295.\n",
      "\n",
      "Intercept: 1.1203 (0.0164).\n",
      "\n",
      "Total Observed scale h2:0.1129 (0.0047).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0148 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: 0.0315 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping RWMm4e, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1079.\n",
      "\n",
      "Intercept: 0.9026(0.0079).\n",
      "\n",
      "Total Observed scale h2:0.0954(0.0091).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4481.\n",
      "\n",
      "Intercept: 1.0378 (0.0097).\n",
      "\n",
      "Total Observed scale h2:0.0736 (0.0036).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 7e-04 (0.0047).\n",
      "\n",
      "Total Observed scale gencov: -0.0078 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bmczy8, 1312 variants, using EUR population reference\n",
      "\n",
      "Removing 1213 of 1312 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1155.\n",
      "\n",
      "Intercept: 0.904(0.0086).\n",
      "\n",
      "Total Observed scale h2:0.0952(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1372.\n",
      "\n",
      "Intercept: 0.9762 (0.0078).\n",
      "\n",
      "Total Observed scale h2:0.0233 (0.002).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0088 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0017 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZLvXKm, 1271 variants, using EUR population reference\n",
      "\n",
      "Removing 1175 of 1271 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1171.\n",
      "\n",
      "Intercept: 0.9036(0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0959(0.0096).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3186.\n",
      "\n",
      "Intercept: 0.9735 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0687 (0.0041).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.002 (0.0056).\n",
      "\n",
      "Total Observed scale gencov: 0.0153 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping V4SVti, 1194 variants, using EUR population reference\n",
      "\n",
      "Removing 1100 of 1194 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.132.\n",
      "\n",
      "Intercept: 0.907(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0948(0.0101).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3039.\n",
      "\n",
      "Intercept: 1.0274 (0.0113).\n",
      "\n",
      "Total Observed scale h2:0.178 (0.0134).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0103 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0146 (0.0057).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping NU4fT4, 1163 variants, using EUR population reference\n",
      "\n",
      "Removing 1071 of 1163 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1188.\n",
      "\n",
      "Intercept: 0.9054(0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0942(0.0095).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4235.\n",
      "\n",
      "Intercept: 1.1214 (0.0131).\n",
      "\n",
      "Total Observed scale h2:0.3525 (0.0334).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0059 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -0.014 (0.0084).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Dk7sEZ, 1168 variants, using EUR population reference\n",
      "\n",
      "Removing 1075 of 1168 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1175.\n",
      "\n",
      "Intercept: 0.9063(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0942(0.0094).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0026.\n",
      "\n",
      "Intercept: 1.346 (0.0222).\n",
      "\n",
      "Total Observed scale h2:0.309 (0.0139).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.005 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0011 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping vULKDN, 1246 variants, using EUR population reference\n",
      "\n",
      "Removing 1150 of 1246 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1182.\n",
      "\n",
      "Intercept: 0.9039(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0957(0.0097).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.5985.\n",
      "\n",
      "Intercept: 1.0344 (0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0604 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0106 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0056 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nyoabH, 1247 variants, using EUR population reference\n",
      "\n",
      "Removing 1150 of 1247 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.9041(0.009).\n",
      "\n",
      "Total Observed scale h2:0.0952(0.0093).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7508.\n",
      "\n",
      "Intercept: 1.0411 (0.0124).\n",
      "\n",
      "Total Observed scale h2:0.1096 (0.0043).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0072 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: 0.0023 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping piT9gW, 1225 variants, using EUR population reference\n",
      "\n",
      "Removing 1132 of 1225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Urate ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1179.\n",
      "\n",
      "Intercept: 0.9052(0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0951(0.0096).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1557.\n",
      "\n",
      "Intercept: 0.9964 (0.0077).\n",
      "\n",
      "Total Observed scale h2:0.0259 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0067 (0.0053).\n",
      "\n",
      "Total Observed scale gencov: -5e-04 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping kphVmD, 1277 variants, using EUR population reference\n",
      "\n",
      "Removing 1182 of 1277 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3069.\n",
      "\n",
      "Intercept: 1.0273(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.1784(0.0131).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1167.\n",
      "\n",
      "Intercept: 1.0385 (0.0118).\n",
      "\n",
      "Total Observed scale h2:0.0661 (0.0142).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.01 (0.0063).\n",
      "\n",
      "Total Observed scale gencov: -1e-04 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mr7KTS, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 967 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3152.\n",
      "\n",
      "Intercept: 1.0249(0.012).\n",
      "\n",
      "Total Observed scale h2:0.1808(0.014).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0883.\n",
      "\n",
      "Intercept: 0.9834 (0.0105).\n",
      "\n",
      "Total Observed scale h2:0.2949 (0.0446).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0117 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0055 (0.0148).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YzGXiv, 1092 variants, using EUR population reference\n",
      "\n",
      "Removing 976 of 1092 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3165.\n",
      "\n",
      "Intercept: 1.0254(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.1807(0.0137).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.5179.\n",
      "\n",
      "Intercept: 1.0389 (0.0141).\n",
      "\n",
      "Total Observed scale h2:0.0476 (0.003).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0107 (0.008).\n",
      "\n",
      "Total Observed scale gencov: -0.0057 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mAdOe2, 1088 variants, using EUR population reference\n",
      "\n",
      "Removing 974 of 1088 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.318.\n",
      "\n",
      "Intercept: 1.0238(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1821(0.0145).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.8141.\n",
      "\n",
      "Intercept: 2.0605 (0.048).\n",
      "\n",
      "Total Observed scale h2:0.4134 (0.0186).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 4e-04 (0.0118).\n",
      "\n",
      "Total Observed scale gencov: 0.0017 (0.0063).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping S9q57W, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0256(0.0118).\n",
      "\n",
      "Total Observed scale h2:0.1807(0.0131).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7422.\n",
      "\n",
      "Intercept: 1.1063 (0.0168).\n",
      "\n",
      "Total Observed scale h2:0.2213 (0.0121).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0089 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: 0.0157 (0.0076).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping DobVM6, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3181.\n",
      "\n",
      "Intercept: 1.0246(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1816(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1274.\n",
      "\n",
      "Intercept: 0.9638 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1262 (0.0153).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0014 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -2e-04 (0.0075).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping DR61g4, 1084 variants, using EUR population reference\n",
      "\n",
      "Removing 970 of 1084 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3167.\n",
      "\n",
      "Intercept: 1.023(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1823(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.085.\n",
      "\n",
      "Intercept: 1.1174 (0.0209).\n",
      "\n",
      "Total Observed scale h2:0.1872 (0.0083).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0127 (0.0095).\n",
      "\n",
      "Total Observed scale gencov: -0.0354 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xSJH5X, 1079 variants, using EUR population reference\n",
      "\n",
      "Removing 966 of 1079 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.315.\n",
      "\n",
      "Intercept: 1.024(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1816(0.0137).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6693.\n",
      "\n",
      "Intercept: 1.0269 (0.0133).\n",
      "\n",
      "Total Observed scale h2:0.0687 (0.0034).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0076 (0.009).\n",
      "\n",
      "Total Observed scale gencov: -0.0043 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MU6GUS, 1093 variants, using EUR population reference\n",
      "\n",
      "Removing 978 of 1093 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0298(0.012).\n",
      "\n",
      "Total Observed scale h2:0.1777(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.1145.\n",
      "\n",
      "Intercept: 1.2173 (0.0256).\n",
      "\n",
      "Total Observed scale h2:0.2154 (0.0071).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0177 (0.0107).\n",
      "\n",
      "Total Observed scale gencov: -0.06 (0.0056).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xTKPty, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3186.\n",
      "\n",
      "Intercept: 1.0229(0.0122).\n",
      "\n",
      "Total Observed scale h2:0.1825(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4183.\n",
      "\n",
      "Intercept: 1.0104 (0.0125).\n",
      "\n",
      "Total Observed scale h2:0.0485 (0.0031).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0082 (0.0082).\n",
      "\n",
      "Total Observed scale gencov: 0.0218 (0.004).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dnlNKF, 1084 variants, using EUR population reference\n",
      "\n",
      "Removing 971 of 1084 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3168.\n",
      "\n",
      "Intercept: 1.0254(0.012).\n",
      "\n",
      "Total Observed scale h2:0.1805(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.5289.\n",
      "\n",
      "Intercept: 1.1588 (0.0162).\n",
      "\n",
      "Total Observed scale h2:0.2688 (0.0248).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0138 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: -0.0058 (0.0084).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping QfqF6B, 1015 variants, using EUR population reference\n",
      "\n",
      "Removing 902 of 1015 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.024(0.0118).\n",
      "\n",
      "Total Observed scale h2:0.1818(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0735.\n",
      "\n",
      "Intercept: 0.9922 (0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0154 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0064 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: 0.0174 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PIxNVy, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3192.\n",
      "\n",
      "Intercept: 1.028(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.18(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0954.\n",
      "\n",
      "Intercept: 1.139 (0.0187).\n",
      "\n",
      "Total Observed scale h2:0.3953 (0.0168).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0374 (0.0109).\n",
      "\n",
      "Total Observed scale gencov: 0.0679 (0.0091).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping avAXNn, 1087 variants, using EUR population reference\n",
      "\n",
      "Removing 973 of 1087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0241(0.0119).\n",
      "\n",
      "Total Observed scale h2:0.1818(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.185.\n",
      "\n",
      "Intercept: 1.0387 (0.0104).\n",
      "\n",
      "Total Observed scale h2:0.0229 (0.0029).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0034 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -0.0034 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping MLnMvv, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3123.\n",
      "\n",
      "Intercept: 1.0256(0.0116).\n",
      "\n",
      "Total Observed scale h2:0.1799(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1631.\n",
      "\n",
      "Intercept: 0.8954 (0.0124).\n",
      "\n",
      "Total Observed scale h2:0.0657 (0.0053).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0065 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: -0.0067 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yx2yAS, 1096 variants, using EUR population reference\n",
      "\n",
      "Removing 978 of 1096 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0243(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1816(0.0135).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.1292.\n",
      "\n",
      "Intercept: 1.1368 (0.0208).\n",
      "\n",
      "Total Observed scale h2:0.114 (0.0053).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0019 (0.0097).\n",
      "\n",
      "Total Observed scale gencov: -0.0138 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping I1GyHg, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.306.\n",
      "\n",
      "Intercept: 1.0269(0.0103).\n",
      "\n",
      "Total Observed scale h2:0.1784(0.0123).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.5016.\n",
      "\n",
      "Intercept: 1.0515 (0.0122).\n",
      "\n",
      "Total Observed scale h2:0.0721 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.001 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: 0.0172 (0.0042).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dNb0dm, 1100 variants, using EUR population reference\n",
      "\n",
      "Removing 981 of 1100 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3108.\n",
      "\n",
      "Intercept: 1.0233(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.1812(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1539.\n",
      "\n",
      "Intercept: 0.975 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0238 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -3e-04 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0037 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping dyUqGQ, 964 variants, using EUR population reference\n",
      "\n",
      "Removing 854 of 964 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3173.\n",
      "\n",
      "Intercept: 1.021(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1848(0.0142).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3578.\n",
      "\n",
      "Intercept: 0.9708 (0.0124).\n",
      "\n",
      "Total Observed scale h2:0.0699 (0.0042).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0038 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0036 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping RtrJcv, 1066 variants, using EUR population reference\n",
      "\n",
      "Removing 954 of 1066 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3039.\n",
      "\n",
      "Intercept: 1.0274(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.178(0.0134).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.132.\n",
      "\n",
      "Intercept: 0.907 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0948 (0.0101).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0103 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0146 (0.0057).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 5KYT8I, 972 variants, using EUR population reference\n",
      "\n",
      "Removing 858 of 972 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3167.\n",
      "\n",
      "Intercept: 1.025(0.0119).\n",
      "\n",
      "Total Observed scale h2:0.1808(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4677.\n",
      "\n",
      "Intercept: 1.1265 (0.0155).\n",
      "\n",
      "Total Observed scale h2:0.367 (0.0361).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0078 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0019 (0.0101).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping S6KpbX, 1015 variants, using EUR population reference\n",
      "\n",
      "Removing 902 of 1015 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3133.\n",
      "\n",
      "Intercept: 1.0199(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.1841(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.1952.\n",
      "\n",
      "Intercept: 1.4074 (0.03).\n",
      "\n",
      "Total Observed scale h2:0.3048 (0.0154).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0024 (0.0105).\n",
      "\n",
      "Total Observed scale gencov: -0.0038 (0.0063).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mo0FyJ, 961 variants, using EUR population reference\n",
      "\n",
      "Removing 853 of 961 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3166.\n",
      "\n",
      "Intercept: 1.0263(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.18(0.0136).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6625.\n",
      "\n",
      "Intercept: 1.0349 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.0614 (0.0029).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0051 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: 0.0058 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mStlZ3, 1088 variants, using EUR population reference\n",
      "\n",
      "Removing 974 of 1088 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0246(0.0118).\n",
      "\n",
      "Total Observed scale h2:0.1814(0.0132).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8211.\n",
      "\n",
      "Intercept: 1.0406 (0.0165).\n",
      "\n",
      "Total Observed scale h2:0.1112 (0.0048).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0225 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: 0.0391 (0.0051).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping s0HHcd, 1086 variants, using EUR population reference\n",
      "\n",
      "Removing 972 of 1086 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anorexia ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3129.\n",
      "\n",
      "Intercept: 1.0265(0.0122).\n",
      "\n",
      "Total Observed scale h2:0.1797(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1695.\n",
      "\n",
      "Intercept: 0.9947 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0262 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0069 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0149 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping HfrnBX, 960 variants, using EUR population reference\n",
      "\n",
      "Removing 851 of 960 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4322.\n",
      "\n",
      "Intercept: 1.1207(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.3604(0.0324).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1125.\n",
      "\n",
      "Intercept: 1.0362 (0.0097).\n",
      "\n",
      "Total Observed scale h2:0.0674 (0.0134).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0072 (0.0061).\n",
      "\n",
      "Total Observed scale gencov: 1e-04 (0.0088).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping KBpHFK, 4320 variants, using EUR population reference\n",
      "\n",
      "Removing 4027 of 4320 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4357.\n",
      "\n",
      "Intercept: 1.12(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.3649(0.0338).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.9716 (0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3203 (0.0376).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0059 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0298 (0.019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 0UYXLf, 4407 variants, using EUR population reference\n",
      "\n",
      "Removing 4115 of 4407 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4348.\n",
      "\n",
      "Intercept: 1.1218(0.012).\n",
      "\n",
      "Total Observed scale h2:0.3614(0.033).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4833.\n",
      "\n",
      "Intercept: 1.0336 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0479 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0074 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0079 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6tZbfO, 4413 variants, using EUR population reference\n",
      "\n",
      "Removing 4118 of 4413 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4314.\n",
      "\n",
      "Intercept: 1.127(0.012).\n",
      "\n",
      "Total Observed scale h2:0.3517(0.0327).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4666.\n",
      "\n",
      "Intercept: 1.9705 (0.04).\n",
      "\n",
      "Total Observed scale h2:0.4175 (0.0178).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0211 (0.0099).\n",
      "\n",
      "Total Observed scale gencov: -0.0072 (0.0099).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XqHmRJ, 4399 variants, using EUR population reference\n",
      "\n",
      "Removing 4104 of 4399 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1228(0.012).\n",
      "\n",
      "Total Observed scale h2:0.3591(0.0318).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6906.\n",
      "\n",
      "Intercept: 1.1075 (0.013).\n",
      "\n",
      "Total Observed scale h2:0.2196 (0.0111).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.013 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0305 (0.0098).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SOmaof, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4315.\n",
      "\n",
      "Intercept: 1.1213(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.3594(0.0329).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1119.\n",
      "\n",
      "Intercept: 0.9578 (0.0097).\n",
      "\n",
      "Total Observed scale h2:0.1275 (0.0138).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0188 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -0.026 (0.0107).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YtjZi1, 4384 variants, using EUR population reference\n",
      "\n",
      "Removing 4091 of 4384 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4345.\n",
      "\n",
      "Intercept: 1.1219(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.3595(0.0337).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0103.\n",
      "\n",
      "Intercept: 1.1122 (0.0184).\n",
      "\n",
      "Total Observed scale h2:0.185 (0.0074).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.026 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: -0.0103 (0.0076).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1rj9Qa, 4342 variants, using EUR population reference\n",
      "\n",
      "Removing 4050 of 4342 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4336.\n",
      "\n",
      "Intercept: 1.1212(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.3613(0.0317).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6285.\n",
      "\n",
      "Intercept: 1.0202 (0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0693 (0.0033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0056 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0043 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YtnBRZ, 4419 variants, using EUR population reference\n",
      "\n",
      "Removing 4124 of 4419 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4328.\n",
      "\n",
      "Intercept: 1.1236(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.358(0.0334).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.938.\n",
      "\n",
      "Intercept: 1.1799 (0.0192).\n",
      "\n",
      "Total Observed scale h2:0.2166 (0.0067).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0028 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: -0.0189 (0.0066).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PJBtGL, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4353.\n",
      "\n",
      "Intercept: 1.1213(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.3626(0.0343).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3822.\n",
      "\n",
      "Intercept: 1.019 (0.0102).\n",
      "\n",
      "Total Observed scale h2:0.0464 (0.0029).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -1e-04 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -0.016 (0.0045).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6PuChH, 4412 variants, using EUR population reference\n",
      "\n",
      "Removing 4117 of 4412 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4271.\n",
      "\n",
      "Intercept: 1.1228(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.3527(0.031).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4842.\n",
      "\n",
      "Intercept: 1.1558 (0.0123).\n",
      "\n",
      "Total Observed scale h2:0.2576 (0.0215).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.8634 (0.0101).\n",
      "\n",
      "Total Observed scale gencov: 0.275 (0.0232).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bZ2IMZ, 4326 variants, using EUR population reference\n",
      "\n",
      "Removing 4031 of 4326 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1222(0.0119).\n",
      "\n",
      "Total Observed scale h2:0.36(0.0324).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0671.\n",
      "\n",
      "Intercept: 0.9849 (0.0073).\n",
      "\n",
      "Total Observed scale h2:0.0168 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0127 (0.0062).\n",
      "\n",
      "Total Observed scale gencov: -0.0152 (0.0043).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 3Mh31T, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4346.\n",
      "\n",
      "Intercept: 1.1195(0.0134).\n",
      "\n",
      "Total Observed scale h2:0.3607(0.0347).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0116.\n",
      "\n",
      "Intercept: 1.095 (0.0142).\n",
      "\n",
      "Total Observed scale h2:0.4071 (0.0151).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0111 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0549 (0.0107).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping b5FIXe, 4295 variants, using EUR population reference\n",
      "\n",
      "Removing 4007 of 4295 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1222(0.012).\n",
      "\n",
      "Total Observed scale h2:0.3599(0.0326).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1714.\n",
      "\n",
      "Intercept: 1.0298 (0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0239 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 6e-04 (0.006).\n",
      "\n",
      "Total Observed scale gencov: -0.0168 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XGXBpR, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1221(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.3601(0.0333).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0424.\n",
      "\n",
      "Intercept: 1.1275 (0.0165).\n",
      "\n",
      "Total Observed scale h2:0.113 (0.0045).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0069 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0167 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping RSTmJN, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1225(0.012).\n",
      "\n",
      "Total Observed scale h2:0.3595(0.0318).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4803.\n",
      "\n",
      "Intercept: 1.0474 (0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0728 (0.0036).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0065 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0194 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6MuMiP, 4424 variants, using EUR population reference\n",
      "\n",
      "Removing 4129 of 4424 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.427.\n",
      "\n",
      "Intercept: 1.1202(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.3567(0.0355).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1426.\n",
      "\n",
      "Intercept: 0.9771 (0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0233 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0041 (0.0064).\n",
      "\n",
      "Total Observed scale gencov: -0.0026 (0.0047).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping gcRVx9, 3847 variants, using EUR population reference\n",
      "\n",
      "Removing 3564 of 3847 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4352.\n",
      "\n",
      "Intercept: 1.1206(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.3631(0.0341).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3303.\n",
      "\n",
      "Intercept: 0.9757 (0.0103).\n",
      "\n",
      "Total Observed scale h2:0.0684 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0151 (0.0072).\n",
      "\n",
      "Total Observed scale gencov: -0.0204 (0.0056).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JoHTcj, 4300 variants, using EUR population reference\n",
      "\n",
      "Removing 4010 of 4300 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4235.\n",
      "\n",
      "Intercept: 1.1214(0.0131).\n",
      "\n",
      "Total Observed scale h2:0.3525(0.0334).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1188.\n",
      "\n",
      "Intercept: 0.9054 (0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0942 (0.0095).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0059 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -0.014 (0.0084).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JrFffB, 3870 variants, using EUR population reference\n",
      "\n",
      "Removing 3585 of 3870 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4677.\n",
      "\n",
      "Intercept: 1.1265(0.0155).\n",
      "\n",
      "Total Observed scale h2:0.367(0.0361).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3167.\n",
      "\n",
      "Intercept: 1.025 (0.0119).\n",
      "\n",
      "Total Observed scale h2:0.1808 (0.0132).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0078 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0019 (0.0101).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 764i9l, 4113 variants, using EUR population reference\n",
      "\n",
      "Removing 3847 of 4113 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4297.\n",
      "\n",
      "Intercept: 1.1206(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.3565(0.0334).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0423.\n",
      "\n",
      "Intercept: 1.3539 (0.0255).\n",
      "\n",
      "Total Observed scale h2:0.3072 (0.0141).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0032 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: 0.0016 (0.0093).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping csym9I, 3826 variants, using EUR population reference\n",
      "\n",
      "Removing 3545 of 3826 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4348.\n",
      "\n",
      "Intercept: 1.1221(0.0122).\n",
      "\n",
      "Total Observed scale h2:0.3609(0.0335).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6121.\n",
      "\n",
      "Intercept: 1.0351 (0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0604 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0185 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0041 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Ne46fx, 4413 variants, using EUR population reference\n",
      "\n",
      "Removing 4118 of 4413 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1224(0.0121).\n",
      "\n",
      "Total Observed scale h2:0.3597(0.0327).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7519.\n",
      "\n",
      "Intercept: 1.0454 (0.0133).\n",
      "\n",
      "Total Observed scale h2:0.1086 (0.0046).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0088 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0159 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping iFaj40, 4426 variants, using EUR population reference\n",
      "\n",
      "Removing 4131 of 4426 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CD ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4259.\n",
      "\n",
      "Intercept: 1.1201(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.3556(0.036).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1595.\n",
      "\n",
      "Intercept: 0.9964 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.026 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0037 (0.006).\n",
      "\n",
      "Total Observed scale gencov: 0.0013 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping aJ52eE, 3853 variants, using EUR population reference\n",
      "\n",
      "Removing 3570 of 3853 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9989.\n",
      "\n",
      "Intercept: 1.3444(0.0224).\n",
      "\n",
      "Total Observed scale h2:0.3086(0.0138).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1103.\n",
      "\n",
      "Intercept: 1.0364 (0.009).\n",
      "\n",
      "Total Observed scale h2:0.0667 (0.0126).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0064 (0.0077).\n",
      "\n",
      "Total Observed scale gencov: -0.0182 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping oZi2xi, 26827 variants, using EUR population reference\n",
      "\n",
      "Removing 25634 of 26827 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0184.\n",
      "\n",
      "Intercept: 1.3505(0.0237).\n",
      "\n",
      "Total Observed scale h2:0.3099(0.0141).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0779.\n",
      "\n",
      "Intercept: 0.9688 (0.008).\n",
      "\n",
      "Total Observed scale h2:0.335 (0.0376).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0154 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0072 (0.0099).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PrMdFu, 27123 variants, using EUR population reference\n",
      "\n",
      "Removing 25926 of 27123 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.025.\n",
      "\n",
      "Intercept: 1.356(0.024).\n",
      "\n",
      "Total Observed scale h2:0.3082(0.014).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4789.\n",
      "\n",
      "Intercept: 1.0331 (0.011).\n",
      "\n",
      "Total Observed scale h2:0.048 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0061 (0.0088).\n",
      "\n",
      "Total Observed scale gencov: -0.0029 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YwAUay, 27174 variants, using EUR population reference\n",
      "\n",
      "Removing 25977 of 27174 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9801.\n",
      "\n",
      "Intercept: 1.359(0.0206).\n",
      "\n",
      "Total Observed scale h2:0.2971(0.0133).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.444.\n",
      "\n",
      "Intercept: 1.9735 (0.0409).\n",
      "\n",
      "Total Observed scale h2:0.4114 (0.0175).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.5155 (0.0255).\n",
      "\n",
      "Total Observed scale gencov: 0.3437 (0.0144).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping OVTyYA, 26390 variants, using EUR population reference\n",
      "\n",
      "Removing 25200 of 26390 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3525(0.0245).\n",
      "\n",
      "Total Observed scale h2:0.3085(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.695.\n",
      "\n",
      "Intercept: 1.115 (0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2171 (0.0102).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0845 (0.009).\n",
      "\n",
      "Total Observed scale gencov: 0.0233 (0.0051).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JLM1dF, 26716 variants, using EUR population reference\n",
      "\n",
      "Removing 25527 of 26716 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0506.\n",
      "\n",
      "Intercept: 1.353(0.0248).\n",
      "\n",
      "Total Observed scale h2:0.3084(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1113.\n",
      "\n",
      "Intercept: 0.9598 (0.0099).\n",
      "\n",
      "Total Observed scale h2:0.1247 (0.0133).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0025 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0076 (0.0059).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ggGd0k, 26592 variants, using EUR population reference\n",
      "\n",
      "Removing 25405 of 26592 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.021.\n",
      "\n",
      "Intercept: 1.3513(0.0229).\n",
      "\n",
      "Total Observed scale h2:0.3099(0.0142).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9836.\n",
      "\n",
      "Intercept: 1.1053 (0.0195).\n",
      "\n",
      "Total Observed scale h2:0.1843 (0.0078).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.008 (0.0095).\n",
      "\n",
      "Total Observed scale gencov: -0.0114 (0.005).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bkkmee, 26621 variants, using EUR population reference\n",
      "\n",
      "Removing 25425 of 26621 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0147.\n",
      "\n",
      "Intercept: 1.3475(0.0231).\n",
      "\n",
      "Total Observed scale h2:0.3097(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6169.\n",
      "\n",
      "Intercept: 1.0208 (0.0118).\n",
      "\n",
      "Total Observed scale h2:0.0692 (0.003).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0107 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.0061 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping IX6YdV, 27149 variants, using EUR population reference\n",
      "\n",
      "Removing 25949 of 27149 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0499.\n",
      "\n",
      "Intercept: 1.3533(0.0246).\n",
      "\n",
      "Total Observed scale h2:0.3083(0.0144).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.944.\n",
      "\n",
      "Intercept: 1.1932 (0.0218).\n",
      "\n",
      "Total Observed scale h2:0.2146 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0088 (0.0117).\n",
      "\n",
      "Total Observed scale gencov: -0.0295 (0.0046).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jMylWs, 26712 variants, using EUR population reference\n",
      "\n",
      "Removing 25524 of 26712 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0546.\n",
      "\n",
      "Intercept: 1.3515(0.0249).\n",
      "\n",
      "Total Observed scale h2:0.3092(0.0141).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3845.\n",
      "\n",
      "Intercept: 1.0083 (0.011).\n",
      "\n",
      "Total Observed scale h2:0.0482 (0.003).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0098 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0076 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping OtGZUE, 26680 variants, using EUR population reference\n",
      "\n",
      "Removing 25494 of 26680 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0441.\n",
      "\n",
      "Intercept: 1.3545(0.0256).\n",
      "\n",
      "Total Observed scale h2:0.3071(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4856.\n",
      "\n",
      "Intercept: 1.1607 (0.0127).\n",
      "\n",
      "Total Observed scale h2:0.2531 (0.0233).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.004 (0.0103).\n",
      "\n",
      "Total Observed scale gencov: -0.0059 (0.0081).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping BseaJK, 25777 variants, using EUR population reference\n",
      "\n",
      "Removing 24603 of 25777 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3525(0.0245).\n",
      "\n",
      "Total Observed scale h2:0.3085(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0686.\n",
      "\n",
      "Intercept: 0.9839 (0.0075).\n",
      "\n",
      "Total Observed scale h2:0.0172 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0033 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -7e-04 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping BUUaWz, 26716 variants, using EUR population reference\n",
      "\n",
      "Removing 25527 of 26716 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0274.\n",
      "\n",
      "Intercept: 1.3459(0.0252).\n",
      "\n",
      "Total Observed scale h2:0.3107(0.0145).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9916.\n",
      "\n",
      "Intercept: 1.1011 (0.0147).\n",
      "\n",
      "Total Observed scale h2:0.403 (0.0155).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.047 (0.0088).\n",
      "\n",
      "Total Observed scale gencov: 1e-04 (0.006).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pYB9O5, 26615 variants, using EUR population reference\n",
      "\n",
      "Removing 25439 of 26615 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3525(0.0245).\n",
      "\n",
      "Total Observed scale h2:0.3085(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1729.\n",
      "\n",
      "Intercept: 1.0364 (0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0229 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0213 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: -0.0125 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping lWvOv6, 26716 variants, using EUR population reference\n",
      "\n",
      "Removing 25527 of 26716 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0105.\n",
      "\n",
      "Intercept: 1.3451(0.0225).\n",
      "\n",
      "Total Observed scale h2:0.31(0.014).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1369.\n",
      "\n",
      "Intercept: 0.8932 (0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0644 (0.005).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0351 (0.0079).\n",
      "\n",
      "Total Observed scale gencov: -0.0122 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping OujMl5, 27285 variants, using EUR population reference\n",
      "\n",
      "Removing 26085 of 27285 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3529(0.0245).\n",
      "\n",
      "Total Observed scale h2:0.3084(0.0142).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0462.\n",
      "\n",
      "Intercept: 1.126 (0.0177).\n",
      "\n",
      "Total Observed scale h2:0.1131 (0.0048).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0096 (0.01).\n",
      "\n",
      "Total Observed scale gencov: -0.0133 (0.0036).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping FDS4q8, 26716 variants, using EUR population reference\n",
      "\n",
      "Removing 25527 of 26716 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9987.\n",
      "\n",
      "Intercept: 1.3401(0.0221).\n",
      "\n",
      "Total Observed scale h2:0.3103(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.474.\n",
      "\n",
      "Intercept: 1.0485 (0.011).\n",
      "\n",
      "Total Observed scale h2:0.0732 (0.0038).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0192 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: 0.0262 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ojtI7r, 27240 variants, using EUR population reference\n",
      "\n",
      "Removing 26039 of 27240 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9997.\n",
      "\n",
      "Intercept: 1.3418(0.0225).\n",
      "\n",
      "Total Observed scale h2:0.3102(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1399.\n",
      "\n",
      "Intercept: 0.9773 (0.0079).\n",
      "\n",
      "Total Observed scale h2:0.0233 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0124 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.0029 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping iNtkVU, 27300 variants, using EUR population reference\n",
      "\n",
      "Removing 26099 of 27300 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0242.\n",
      "\n",
      "Intercept: 1.3516(0.0239).\n",
      "\n",
      "Total Observed scale h2:0.311(0.0141).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3272.\n",
      "\n",
      "Intercept: 0.9753 (0.0095).\n",
      "\n",
      "Total Observed scale h2:0.0692 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0205 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: -0.0017 (0.0031).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9XBu0O, 26221 variants, using EUR population reference\n",
      "\n",
      "Removing 25029 of 26221 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0026.\n",
      "\n",
      "Intercept: 1.346(0.0222).\n",
      "\n",
      "Total Observed scale h2:0.309(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1175.\n",
      "\n",
      "Intercept: 0.9063 (0.009).\n",
      "\n",
      "Total Observed scale h2:0.0942 (0.0094).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.005 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0011 (0.0049).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping N3EaGL, 27339 variants, using EUR population reference\n",
      "\n",
      "Removing 26139 of 27339 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.1952.\n",
      "\n",
      "Intercept: 1.4074(0.03).\n",
      "\n",
      "Total Observed scale h2:0.3048(0.0154).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3133.\n",
      "\n",
      "Intercept: 1.0199 (0.0129).\n",
      "\n",
      "Total Observed scale h2:0.1841 (0.0143).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0024 (0.0105).\n",
      "\n",
      "Total Observed scale gencov: -0.0038 (0.0063).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping VZAa0M, 24497 variants, using EUR population reference\n",
      "\n",
      "Removing 23351 of 24497 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0423.\n",
      "\n",
      "Intercept: 1.3539(0.0255).\n",
      "\n",
      "Total Observed scale h2:0.3072(0.0141).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4297.\n",
      "\n",
      "Intercept: 1.1206 (0.0129).\n",
      "\n",
      "Total Observed scale h2:0.3565 (0.0334).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0032 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: 0.0016 (0.0093).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping pG9qQz, 25763 variants, using EUR population reference\n",
      "\n",
      "Removing 24588 of 25763 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.025.\n",
      "\n",
      "Intercept: 1.3556(0.0239).\n",
      "\n",
      "Total Observed scale h2:0.3083(0.014).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6072.\n",
      "\n",
      "Intercept: 1.0397 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0601 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0222 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: -0.0071 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping eBjCFp, 27173 variants, using EUR population reference\n",
      "\n",
      "Removing 25976 of 27173 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3528(0.0248).\n",
      "\n",
      "Total Observed scale h2:0.3084(0.0143).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7631.\n",
      "\n",
      "Intercept: 1.0432 (0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1099 (0.0043).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0101 (0.0097).\n",
      "\n",
      "Total Observed scale gencov: -0.011 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping x0ZfzZ, 26716 variants, using EUR population reference\n",
      "\n",
      "Removing 25527 of 26716 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Height_GIANT ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0186.\n",
      "\n",
      "Intercept: 1.3506(0.0235).\n",
      "\n",
      "Total Observed scale h2:0.3095(0.0139).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.159.\n",
      "\n",
      "Intercept: 0.9987 (0.0079).\n",
      "\n",
      "Total Observed scale h2:0.0257 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0064 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0045 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping hgmFGG, 27157 variants, using EUR population reference\n",
      "\n",
      "Removing 25958 of 27157 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6068.\n",
      "\n",
      "Intercept: 1.0345(0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0607(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1122.\n",
      "\n",
      "Intercept: 1.0358 (0.0085).\n",
      "\n",
      "Total Observed scale h2:0.0683 (0.0133).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0049 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: -6e-04 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping rCK4dw, 4000 variants, using EUR population reference\n",
      "\n",
      "Removing 3646 of 4000 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6079.\n",
      "\n",
      "Intercept: 1.0331(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.061(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.077.\n",
      "\n",
      "Intercept: 0.9718 (0.0072).\n",
      "\n",
      "Total Observed scale h2:0.3211 (0.0381).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0078 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -0.0043 (0.0069).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping SJC6jE, 4069 variants, using EUR population reference\n",
      "\n",
      "Removing 3714 of 4069 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6045.\n",
      "\n",
      "Intercept: 1.0369(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0603(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4766.\n",
      "\n",
      "Intercept: 1.0338 (0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0479 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0942 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0123 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mDupD5, 4073 variants, using EUR population reference\n",
      "\n",
      "Removing 3717 of 4073 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6109.\n",
      "\n",
      "Intercept: 1.0375(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.06(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4937.\n",
      "\n",
      "Intercept: 1.9617 (0.0405).\n",
      "\n",
      "Total Observed scale h2:0.4212 (0.0176).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0425 (0.0108).\n",
      "\n",
      "Total Observed scale gencov: -0.0142 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping GZ6E2t, 3983 variants, using EUR population reference\n",
      "\n",
      "Removing 3635 of 3983 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0351(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6948.\n",
      "\n",
      "Intercept: 1.1078 (0.0132).\n",
      "\n",
      "Total Observed scale h2:0.2201 (0.0112).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.006 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0153 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 5LfIsL, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6142.\n",
      "\n",
      "Intercept: 1.0334(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0607(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1128.\n",
      "\n",
      "Intercept: 0.9554 (0.0098).\n",
      "\n",
      "Total Observed scale h2:0.1299 (0.0136).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0017 (0.0066).\n",
      "\n",
      "Total Observed scale gencov: 0.0099 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ERSEbd, 3981 variants, using EUR population reference\n",
      "\n",
      "Removing 3635 of 3981 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6077.\n",
      "\n",
      "Intercept: 1.0383(0.0107).\n",
      "\n",
      "Total Observed scale h2:0.0603(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.996.\n",
      "\n",
      "Intercept: 1.1102 (0.019).\n",
      "\n",
      "Total Observed scale h2:0.1854 (0.0077).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0322 (0.0094).\n",
      "\n",
      "Total Observed scale gencov: 0.0194 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xdStr0, 4002 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 4002 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6056.\n",
      "\n",
      "Intercept: 1.0358(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0605(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.622.\n",
      "\n",
      "Intercept: 1.0254 (0.0114).\n",
      "\n",
      "Total Observed scale h2:0.0689 (0.0033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0222 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0041 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 2XCiv9, 4057 variants, using EUR population reference\n",
      "\n",
      "Removing 3702 of 4057 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0389(0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0599(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9528.\n",
      "\n",
      "Intercept: 1.1844 (0.0198).\n",
      "\n",
      "Total Observed scale h2:0.2169 (0.0067).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0579 (0.0098).\n",
      "\n",
      "Total Observed scale gencov: 0.021 (0.0028).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ARTkh0, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6126.\n",
      "\n",
      "Intercept: 1.034(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0605(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3825.\n",
      "\n",
      "Intercept: 1.0188 (0.0101).\n",
      "\n",
      "Total Observed scale h2:0.0464 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1771 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0241 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ujyeXw, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6119.\n",
      "\n",
      "Intercept: 1.0347(0.0113).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.495.\n",
      "\n",
      "Intercept: 1.1576 (0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2633 (0.0228).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0185 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -0.0056 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JmK77S, 3850 variants, using EUR population reference\n",
      "\n",
      "Removing 3510 of 3850 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0346(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0605(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.068.\n",
      "\n",
      "Intercept: 0.9844 (0.0073).\n",
      "\n",
      "Total Observed scale h2:0.0171 (0.0025).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0431 (0.0058).\n",
      "\n",
      "Total Observed scale gencov: 0.0117 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping AlZX8e, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.613.\n",
      "\n",
      "Intercept: 1.0406(0.0116).\n",
      "\n",
      "Total Observed scale h2:0.0603(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0006.\n",
      "\n",
      "Intercept: 1.0972 (0.0146).\n",
      "\n",
      "Total Observed scale h2:0.4066 (0.0151).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0013 (0.009).\n",
      "\n",
      "Total Observed scale gencov: 0.0044 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping fEcXl5, 4028 variants, using EUR population reference\n",
      "\n",
      "Removing 3678 of 4028 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0344(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0605(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1728.\n",
      "\n",
      "Intercept: 1.0293 (0.0089).\n",
      "\n",
      "Total Observed scale h2:0.0241 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0283 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.012 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 6TBw04, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6057.\n",
      "\n",
      "Intercept: 1.035(0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0606(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.143.\n",
      "\n",
      "Intercept: 0.8948 (0.0093).\n",
      "\n",
      "Total Observed scale h2:0.0652 (0.005).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0038 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: 0.0105 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping nQhaDY, 4070 variants, using EUR population reference\n",
      "\n",
      "Removing 3716 of 4070 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0347(0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0605(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0528.\n",
      "\n",
      "Intercept: 1.1265 (0.017).\n",
      "\n",
      "Total Observed scale h2:0.114 (0.0047).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0582 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: 0.0176 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 0Banj3, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6046.\n",
      "\n",
      "Intercept: 1.037(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0603(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4782.\n",
      "\n",
      "Intercept: 1.0479 (0.0109).\n",
      "\n",
      "Total Observed scale h2:0.0733 (0.0038).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0593 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: -0.0184 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JfuVtU, 4059 variants, using EUR population reference\n",
      "\n",
      "Removing 3703 of 4059 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6016.\n",
      "\n",
      "Intercept: 1.0358(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1411.\n",
      "\n",
      "Intercept: 0.9794 (0.0084).\n",
      "\n",
      "Total Observed scale h2:0.0231 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0026 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: 0.0062 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping qICaCA, 3573 variants, using EUR population reference\n",
      "\n",
      "Removing 3226 of 3573 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6074.\n",
      "\n",
      "Intercept: 1.0354(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0607(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3272.\n",
      "\n",
      "Intercept: 0.9745 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.069 (0.004).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0086 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0122 (0.0021).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zH7HvU, 3944 variants, using EUR population reference\n",
      "\n",
      "Removing 3593 of 3944 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.5985.\n",
      "\n",
      "Intercept: 1.0344(0.0105).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1182.\n",
      "\n",
      "Intercept: 0.9039 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0957 (0.0097).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0106 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0056 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 69DWpB, 3586 variants, using EUR population reference\n",
      "\n",
      "Removing 3239 of 3586 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6625.\n",
      "\n",
      "Intercept: 1.0349(0.0143).\n",
      "\n",
      "Total Observed scale h2:0.0614(0.0029).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3166.\n",
      "\n",
      "Intercept: 1.0263 (0.0123).\n",
      "\n",
      "Total Observed scale h2:0.18 (0.0136).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0051 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: 0.0058 (0.0038).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping mjlztE, 3768 variants, using EUR population reference\n",
      "\n",
      "Removing 3451 of 3768 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6121.\n",
      "\n",
      "Intercept: 1.0351(0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0604(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4348.\n",
      "\n",
      "Intercept: 1.1221 (0.0122).\n",
      "\n",
      "Total Observed scale h2:0.3609 (0.0335).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0185 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0041 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 0Oc3Cf, 3851 variants, using EUR population reference\n",
      "\n",
      "Removing 3511 of 3851 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6072.\n",
      "\n",
      "Intercept: 1.0397(0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0601(0.0025).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.025.\n",
      "\n",
      "Intercept: 1.3556 (0.0239).\n",
      "\n",
      "Total Observed scale h2:0.3083 (0.014).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0222 (0.0086).\n",
      "\n",
      "Total Observed scale gencov: -0.0071 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping j2BZec, 3575 variants, using EUR population reference\n",
      "\n",
      "Removing 3230 of 3575 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ Neuroticism \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0362(0.011).\n",
      "\n",
      "Total Observed scale h2:0.0603(0.0023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7589.\n",
      "\n",
      "Intercept: 1.0475 (0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1087 (0.0044).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.2282 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.034 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping yJCjFQ, 3997 variants, using EUR population reference\n",
      "\n",
      "Removing 3649 of 3997 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Insomnia ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6008.\n",
      "\n",
      "Intercept: 1.0333(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.0607(0.0026).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1578.\n",
      "\n",
      "Intercept: 0.9959 (0.0078).\n",
      "\n",
      "Total Observed scale h2:0.0261 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0289 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: -0.0141 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping YrTYcH, 3576 variants, using EUR population reference\n",
      "\n",
      "Removing 3231 of 3576 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7555.\n",
      "\n",
      "Intercept: 1.043(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.1092(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1132.\n",
      "\n",
      "Intercept: 1.038 (0.01).\n",
      "\n",
      "Total Observed scale h2:0.0664 (0.0133).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0149 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0104 (0.0044).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PJq0GZ, 6100 variants, using EUR population reference\n",
      "\n",
      "Removing 5641 of 6100 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7564.\n",
      "\n",
      "Intercept: 1.0469(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1084(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0778.\n",
      "\n",
      "Intercept: 0.972 (0.0075).\n",
      "\n",
      "Total Observed scale h2:0.3186 (0.0378).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0064 (0.0074).\n",
      "\n",
      "Total Observed scale gencov: 0.0207 (0.0092).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xgkQZ3, 6169 variants, using EUR population reference\n",
      "\n",
      "Removing 5711 of 6169 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7589.\n",
      "\n",
      "Intercept: 1.046(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.109(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4832.\n",
      "\n",
      "Intercept: 1.0337 (0.0106).\n",
      "\n",
      "Total Observed scale h2:0.0478 (0.0026).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1386 (0.0078).\n",
      "\n",
      "Total Observed scale gencov: 0.0184 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 395c6g, 6218 variants, using EUR population reference\n",
      "\n",
      "Removing 5755 of 6218 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7537.\n",
      "\n",
      "Intercept: 1.0428(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1093(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4786.\n",
      "\n",
      "Intercept: 1.9695 (0.0398).\n",
      "\n",
      "Total Observed scale h2:0.4186 (0.0177).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0851 (0.0118).\n",
      "\n",
      "Total Observed scale gencov: -0.012 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zLWd9C, 6210 variants, using EUR population reference\n",
      "\n",
      "Removing 5747 of 6210 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0462(0.0125).\n",
      "\n",
      "Total Observed scale h2:0.1086(0.0043).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6916.\n",
      "\n",
      "Intercept: 1.1098 (0.0131).\n",
      "\n",
      "Total Observed scale h2:0.2187 (0.0111).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0408 (0.008).\n",
      "\n",
      "Total Observed scale gencov: -0.0246 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 1yRYSr, 6227 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6227 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7556.\n",
      "\n",
      "Intercept: 1.0432(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1092(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.112.\n",
      "\n",
      "Intercept: 0.9569 (0.0096).\n",
      "\n",
      "Total Observed scale h2:0.1283 (0.0136).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0053 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0069 (0.0041).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping sTRcme, 6199 variants, using EUR population reference\n",
      "\n",
      "Removing 5737 of 6199 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7608.\n",
      "\n",
      "Intercept: 1.0473(0.013).\n",
      "\n",
      "Total Observed scale h2:0.1088(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0099.\n",
      "\n",
      "Intercept: 1.112 (0.0184).\n",
      "\n",
      "Total Observed scale h2:0.185 (0.0074).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0155 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: 0.0081 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping r4LTQ0, 6093 variants, using EUR population reference\n",
      "\n",
      "Removing 5632 of 6093 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7554.\n",
      "\n",
      "Intercept: 1.0453(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1087(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6281.\n",
      "\n",
      "Intercept: 1.02 (0.0112).\n",
      "\n",
      "Total Observed scale h2:0.0693 (0.0033).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0046 (0.0082).\n",
      "\n",
      "Total Observed scale gencov: -0.0026 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping bfg5pb, 6198 variants, using EUR population reference\n",
      "\n",
      "Removing 5735 of 6198 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.754.\n",
      "\n",
      "Intercept: 1.0489(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.108(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9412.\n",
      "\n",
      "Intercept: 1.1805 (0.0193).\n",
      "\n",
      "Total Observed scale h2:0.2167 (0.0068).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0057 (0.0096).\n",
      "\n",
      "Total Observed scale gencov: 7e-04 (0.003).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping m4ObjM, 6223 variants, using EUR population reference\n",
      "\n",
      "Removing 5760 of 6223 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7598.\n",
      "\n",
      "Intercept: 1.0466(0.013).\n",
      "\n",
      "Total Observed scale h2:0.109(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3826.\n",
      "\n",
      "Intercept: 1.0214 (0.0099).\n",
      "\n",
      "Total Observed scale h2:0.046 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.4556 (0.0092).\n",
      "\n",
      "Total Observed scale gencov: 0.0555 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping E86Umf, 6216 variants, using EUR population reference\n",
      "\n",
      "Removing 5754 of 6216 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7523.\n",
      "\n",
      "Intercept: 1.0455(0.0134).\n",
      "\n",
      "Total Observed scale h2:0.1086(0.0046).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4924.\n",
      "\n",
      "Intercept: 1.156 (0.0128).\n",
      "\n",
      "Total Observed scale h2:0.2635 (0.0224).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0042 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: -0.019 (0.0048).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping kQTLnO, 5978 variants, using EUR population reference\n",
      "\n",
      "Removing 5517 of 5978 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0459(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1087(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0673.\n",
      "\n",
      "Intercept: 0.9861 (0.0072).\n",
      "\n",
      "Total Observed scale h2:0.0166 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.1303 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: 0.0226 (0.0022).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping RWc1Fp, 6227 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6227 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7649.\n",
      "\n",
      "Intercept: 1.0444(0.0123).\n",
      "\n",
      "Total Observed scale h2:0.1094(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0124.\n",
      "\n",
      "Intercept: 1.0976 (0.0144).\n",
      "\n",
      "Total Observed scale h2:0.4057 (0.015).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0085 (0.0097).\n",
      "\n",
      "Total Observed scale gencov: 0.0451 (0.0055).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ykkVWP, 6087 variants, using EUR population reference\n",
      "\n",
      "Removing 5630 of 6087 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0457(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1087(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1717.\n",
      "\n",
      "Intercept: 1.029 (0.0087).\n",
      "\n",
      "Total Observed scale h2:0.0241 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0428 (0.006).\n",
      "\n",
      "Total Observed scale gencov: 0.0056 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping HisrWx, 6227 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6227 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7557.\n",
      "\n",
      "Intercept: 1.0455(0.0128).\n",
      "\n",
      "Total Observed scale h2:0.1087(0.0045).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1465.\n",
      "\n",
      "Intercept: 0.895 (0.0094).\n",
      "\n",
      "Total Observed scale h2:0.0652 (0.0049).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.009 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: 0.0046 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping sYP7DI, 6225 variants, using EUR population reference\n",
      "\n",
      "Removing 5762 of 6225 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0468(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.1085(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.0462.\n",
      "\n",
      "Intercept: 1.1257 (0.017).\n",
      "\n",
      "Total Observed scale h2:0.1136 (0.0046).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0708 (0.0084).\n",
      "\n",
      "Total Observed scale gencov: 0.0121 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping WK63KB, 6227 variants, using EUR population reference\n",
      "\n",
      "Removing 5764 of 6227 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Income \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1208814\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7542.\n",
      "\n",
      "Intercept: 1.0456(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1087(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4824.\n",
      "\n",
      "Intercept: 1.0476 (0.0105).\n",
      "\n",
      "Total Observed scale h2:0.073 (0.0037).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0925 (0.0081).\n",
      "\n",
      "Total Observed scale gencov: -0.0233 (0.0026).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping tSdMIU, 6209 variants, using EUR population reference\n",
      "\n",
      "Removing 5746 of 6209 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ NEB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m984184\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7556.\n",
      "\n",
      "Intercept: 1.0422(0.0129).\n",
      "\n",
      "Total Observed scale h2:0.1097(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1426.\n",
      "\n",
      "Intercept: 0.9787 (0.0085).\n",
      "\n",
      "Total Observed scale h2:0.023 (0.0021).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0037 (0.0073).\n",
      "\n",
      "Total Observed scale gencov: 0.0023 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping TIWx33, 5464 variants, using EUR population reference\n",
      "\n",
      "Removing 5017 of 5464 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Smoking \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1043420\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.753.\n",
      "\n",
      "Intercept: 1.0442(0.0119).\n",
      "\n",
      "Total Observed scale h2:0.109(0.0043).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3301.\n",
      "\n",
      "Intercept: 0.9745 (0.01).\n",
      "\n",
      "Total Observed scale h2:0.0686 (0.0039).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0108 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: 0.0118 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping jFKRGj, 5899 variants, using EUR population reference\n",
      "\n",
      "Removing 5445 of 5899 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Urate \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1060305\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7508.\n",
      "\n",
      "Intercept: 1.0411(0.0124).\n",
      "\n",
      "Total Observed scale h2:0.1096(0.0043).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1202.\n",
      "\n",
      "Intercept: 0.9041 (0.009).\n",
      "\n",
      "Total Observed scale h2:0.0952 (0.0093).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0072 (0.0065).\n",
      "\n",
      "Total Observed scale gencov: 0.0023 (0.0029).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Z6VszC, 5558 variants, using EUR population reference\n",
      "\n",
      "Removing 5108 of 5558 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Anorexia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m910374\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.8211.\n",
      "\n",
      "Intercept: 1.0406(0.0165).\n",
      "\n",
      "Total Observed scale h2:0.1112(0.0048).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3179.\n",
      "\n",
      "Intercept: 1.0246 (0.0118).\n",
      "\n",
      "Total Observed scale h2:0.1814 (0.0132).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0225 (0.0091).\n",
      "\n",
      "Total Observed scale gencov: 0.0391 (0.0051).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping teGZzm, 5787 variants, using EUR population reference\n",
      "\n",
      "Removing 5357 of 5787 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ CD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004565\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7519.\n",
      "\n",
      "Intercept: 1.0454(0.0133).\n",
      "\n",
      "Total Observed scale h2:0.1086(0.0046).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4329.\n",
      "\n",
      "Intercept: 1.1224 (0.0121).\n",
      "\n",
      "Total Observed scale h2:0.3597 (0.0327).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0088 (0.007).\n",
      "\n",
      "Total Observed scale gencov: -0.0159 (0.0058).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping z2vefz, 5971 variants, using EUR population reference\n",
      "\n",
      "Removing 5510 of 5971 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Height_GIANT \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m961042\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7631.\n",
      "\n",
      "Intercept: 1.0432(0.0126).\n",
      "\n",
      "Total Observed scale h2:0.1099(0.0043).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:3.0501.\n",
      "\n",
      "Intercept: 1.3528 (0.0248).\n",
      "\n",
      "Total Observed scale h2:0.3084 (0.0143).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0101 (0.0097).\n",
      "\n",
      "Total Observed scale gencov: -0.011 (0.0035).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping kEpDac, 5473 variants, using EUR population reference\n",
      "\n",
      "Removing 5030 of 5473 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ Insomnia \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065149\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7589.\n",
      "\n",
      "Intercept: 1.0475(0.0127).\n",
      "\n",
      "Total Observed scale h2:0.1087(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6124.\n",
      "\n",
      "Intercept: 1.0362 (0.011).\n",
      "\n",
      "Total Observed scale h2:0.0603 (0.0023).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.2282 (0.0085).\n",
      "\n",
      "Total Observed scale gencov: 0.034 (0.0023).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping QuygcL, 6218 variants, using EUR population reference\n",
      "\n",
      "Removing 5755 of 6218 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Neuroticism ~ SWB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.7554.\n",
      "\n",
      "Intercept: 1.0421(0.013).\n",
      "\n",
      "Total Observed scale h2:0.1098(0.0044).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.9964 (0.0081).\n",
      "\n",
      "Total Observed scale h2:0.0259 (0.0024).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0496 (0.0075).\n",
      "\n",
      "Total Observed scale gencov: -0.0351 (0.0024).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping Uolbo9, 5518 variants, using EUR population reference\n",
      "\n",
      "Removing 5068 of 5518 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ AD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1148476\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1554.\n",
      "\n",
      "Intercept: 0.9986(0.0074).\n",
      "\n",
      "Total Observed scale h2:0.0254(0.0023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1077.\n",
      "\n",
      "Intercept: 1.031 (0.0076).\n",
      "\n",
      "Total Observed scale h2:0.0698 (0.0123).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0074 (0.006).\n",
      "\n",
      "Total Observed scale gencov: -0.0043 (0.0034).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping zruGKL, 282 variants, using EUR population reference\n",
      "\n",
      "Removing 216 of 282 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ ASD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1076873\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1551.\n",
      "\n",
      "Intercept: 0.9959(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.0259(0.0023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0759.\n",
      "\n",
      "Intercept: 0.9684 (0.007).\n",
      "\n",
      "Total Observed scale h2:0.3327 (0.0366).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 7e-04 (0.0057).\n",
      "\n",
      "Total Observed scale gencov: -0.0187 (0.0066).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping olm60V, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Daytime_Sleepiness \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1065141\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1578.\n",
      "\n",
      "Intercept: 0.9961(0.0078).\n",
      "\n",
      "Total Observed scale h2:0.0261(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4753.\n",
      "\n",
      "Intercept: 1.034 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0479 (0.0027).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0079 (0.0062).\n",
      "\n",
      "Total Observed scale gencov: -0.0076 (0.0014).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 9EJyna, 288 variants, using EUR population reference\n",
      "\n",
      "Removing 223 of 288 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Height_UKB \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037320\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1595.\n",
      "\n",
      "Intercept: 0.9968(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.0258(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:5.4055.\n",
      "\n",
      "Intercept: 1.958 (0.0411).\n",
      "\n",
      "Total Observed scale h2:0.412 (0.0171).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0097 (0.009).\n",
      "\n",
      "Total Observed scale gencov: 0.0025 (0.0027).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping PgBGqn, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Intelligence \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.996(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6874.\n",
      "\n",
      "Intercept: 1.1111 (0.013).\n",
      "\n",
      "Total Observed scale h2:0.2176 (0.0105).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0037 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: 0.0018 (0.0033).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping EhPxCG, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ RA \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1029912\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.159.\n",
      "\n",
      "Intercept: 0.9961(0.0082).\n",
      "\n",
      "Total Observed scale h2:0.0259(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1093.\n",
      "\n",
      "Intercept: 0.9545 (0.0102).\n",
      "\n",
      "Total Observed scale h2:0.1287 (0.0144).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0037 (0.0054).\n",
      "\n",
      "Total Observed scale gencov: 0.0026 (0.0032).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping XPPzMs, 285 variants, using EUR population reference\n",
      "\n",
      "Removing 221 of 285 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ T2D \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1048310\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1574.\n",
      "\n",
      "Intercept: 0.9963(0.0076).\n",
      "\n",
      "Total Observed scale h2:0.0261(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9749.\n",
      "\n",
      "Intercept: 1.1017 (0.0179).\n",
      "\n",
      "Total Observed scale h2:0.1849 (0.0073).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0073 (0.0068).\n",
      "\n",
      "Total Observed scale gencov: -0.0027 (0.0025).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping fJUQhB, 285 variants, using EUR population reference\n",
      "\n",
      "Removing 219 of 285 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Alcohol \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1084534\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1558.\n",
      "\n",
      "Intercept: 0.9958(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.6084.\n",
      "\n",
      "Intercept: 1.0214 (0.0111).\n",
      "\n",
      "Total Observed scale h2:0.0687 (0.0031).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0084 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0018 (0.0017).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping xzF1VW, 289 variants, using EUR population reference\n",
      "\n",
      "Removing 223 of 289 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ BMI \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037799\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1591.\n",
      "\n",
      "Intercept: 0.9975(0.008).\n",
      "\n",
      "Total Observed scale h2:0.0257(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:2.9224.\n",
      "\n",
      "Intercept: 1.1787 (0.02).\n",
      "\n",
      "Total Observed scale h2:0.2157 (0.0065).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: 0.0027 (0.0076).\n",
      "\n",
      "Total Observed scale gencov: -0.0023 (0.002).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping ZyQMJN, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Depression \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1026744\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1603.\n",
      "\n",
      "Intercept: 0.9968(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.0259(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.3824.\n",
      "\n",
      "Intercept: 1.0123 (0.0108).\n",
      "\n",
      "Total Observed scale h2:0.0477 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0476 (0.0069).\n",
      "\n",
      "Total Observed scale gencov: -0.0269 (0.0018).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping X91dL5, 286 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 286 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ IBD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1004597\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1594.\n",
      "\n",
      "Intercept: 0.9964(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.4818.\n",
      "\n",
      "Intercept: 1.1547 (0.0126).\n",
      "\n",
      "Total Observed scale h2:0.2575 (0.0234).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0013 (0.0067).\n",
      "\n",
      "Total Observed scale gencov: 0.0032 (0.0039).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping fpftbw, 281 variants, using EUR population reference\n",
      "\n",
      "Removing 216 of 281 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ MDD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.996(0.0082).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.0676.\n",
      "\n",
      "Intercept: 0.9868 (0.008).\n",
      "\n",
      "Total Observed scale h2:0.0165 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0165 (0.0057).\n",
      "\n",
      "Total Observed scale gencov: -0.0086 (0.0016).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping JkAsPo, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ SCZ \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1047564\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1551.\n",
      "\n",
      "Intercept: 0.997(0.0079).\n",
      "\n",
      "Total Observed scale h2:0.0256(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.9778.\n",
      "\n",
      "Intercept: 1.0943 (0.0143).\n",
      "\n",
      "Total Observed scale h2:0.4041 (0.0152).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0149 (0.0071).\n",
      "\n",
      "Total Observed scale gencov: -0.0312 (0.0037).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping 83tabL, 284 variants, using EUR population reference\n",
      "\n",
      "Removing 219 of 284 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ Angina \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1593.\n",
      "\n",
      "Intercept: 0.996(0.0081).\n",
      "\n",
      "Total Observed scale h2:0.026(0.0024).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1705.\n",
      "\n",
      "Intercept: 1.0313 (0.0091).\n",
      "\n",
      "Total Observed scale h2:0.0236 (0.0028).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0038 (0.006).\n",
      "\n",
      "Total Observed scale gencov: -0.0021 (0.0015).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping KFIVhO, 287 variants, using EUR population reference\n",
      "\n",
      "Removing 222 of 287 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ CAD \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1112977\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1554.\n",
      "\n",
      "Intercept: 0.9964(0.0075).\n",
      "\n",
      "Total Observed scale h2:0.0258(0.0023).\n",
      "\n",
      "\n",
      "Estimate heritability for trait 2 ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Mean Chi2:1.1347.\n",
      "\n",
      "Intercept: 0.8896 (0.0096).\n",
      "\n",
      "Total Observed scale h2:0.0654 (0.0051).\n",
      "\n",
      "\n",
      "Estimate genetic covariance ...\n",
      "\n",
      "Using two-step estimator with cutoff at 30.\n",
      "\n",
      "Intercept: -0.0116 (0.0059).\n",
      "\n",
      "Total Observed scale gencov: 2e-04 (0.0019).\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Begin clumping ...\n",
      " "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Clumping WRj6xX, 289 variants, using EUR population reference\n",
      "\n",
      "Removing 223 of 289 variants due to LD with other variants or absence from LD reference panel\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SWB ~ HBP \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m975109\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "\u001b[1m\u001b[1mRows: \u001b[1m\u001b[22m\u001b[34m\u001b[34m1037824\u001b[34m\u001b[39m \u001b[1m\u001b[1mColumns: \u001b[1m\u001b[22m\u001b[34m\u001b[34m7\u001b[34m\u001b[39m\n",
      "\n",
      "\u001b[36m──\u001b[39m \u001b[1m\u001b[1mColumn specification\u001b[1m\u001b[22m \u001b[36m────────────────────────────────────────────────────────────────────────────────────────────────────────\u001b[39m\n",
      "\u001b[1mDelimiter:\u001b[22m \"\\t\"\n",
      "\u001b[31mchr\u001b[39m (3): SNP, A1, A2\n",
      "\u001b[32mdbl\u001b[39m (4): Z, N, chi2, P\n",
      "\n",
      "\n",
      "\u001b[36mℹ\u001b[39m Use \u001b[30m\u001b[47m\u001b[30m\u001b[47m`spec()`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to retrieve the full column specification for this data.\n",
      "\u001b[36mℹ\u001b[39m Specify the column types or set \u001b[30m\u001b[47m\u001b[30m\u001b[47m`show_col_types = FALSE`\u001b[47m\u001b[30m\u001b[49m\u001b[39m to quiet this message.\n",
      "\n",
      "Merge dat1 and dat2 by SNP ...\n",
      "\n",
      "Harmonize the direction of SNP effects of exposure and outcome\n",
      "\n",
      "Read in LD scores ... \n",
      "\n",
      "Add LD scores to the harmonized data set...\n",
      "\n",
      "The Harmonized data set will also be used for MR analysis \n",
      "\n",
      "\n",
      "Begin estimation of C and Omega using LDSC ...\n",
      "\n",
      "Estimate heritability for trait 1 ...\n",
      "\n",
      "IOPub message rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_msg_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_msg_rate_limit=1000.0 (msgs/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "## Estimating parameters for the background model using genome-wide summary statistics;\n",
    "## using plink clumping to obtain independent IVs\n",
    "\n",
    "ts1 = c(\"AD\", \"ASD\", \"Daytime_Sleepiness\", \"Height_UKB\",  \"Intelligence\", \"RA\",      \n",
    "        \"T2D\", \"Alcohol\", \"BMI\", \"Depression\", \"IBD\", \"MDD\", \"SCZ\", \"Angina\", \n",
    "        \"CAD\", \"HBP\", \"Income\", \"NEB\", \"Smoking\", \"Urate\", \"Anorexia\", \n",
    "        \"CD\", \"Height_GIANT\", \"Insomnia\", \"Neuroticism\", \"SWB\")\n",
    "\n",
    "ts2 = ts1\n",
    "\n",
    "\n",
    "for( exposure in ts1){\n",
    "  for( outcome in ts2){\n",
    "    \n",
    "    if(exposure==outcome) next\n",
    "      \n",
    "    # Start the clock!\n",
    "    start = proc.time()\n",
    "    \n",
    "    # read in formatted GWAS data\n",
    "    trait1.dir = paste0(\"./GWAS_26and5_formatted/\", exposure)\n",
    "    trait2.dir = paste0(\"./GWAS_26and5_formatted/\", outcome)\n",
    "    \n",
    "    \n",
    "    cat(exposure,\"~\",outcome,\"\\n\")\n",
    "    \n",
    "    trait1 = readr::read_delim(trait1.dir, delim = \"\\t\", escape_double = FALSE, trim_ws = TRUE, progress = F)\n",
    "    \n",
    "    trait2 = readr::read_delim(trait2.dir, delim = \"\\t\", escape_double = FALSE, trim_ws = TRUE, progress = F)\n",
    "    \n",
    "    # estimate parameters for the background model\n",
    "    paras = est_paras(dat1 = trait1,\n",
    "                      dat2 = trait2,\n",
    "                      trait1.name = exposure,\n",
    "                      trait2.name = outcome,\n",
    "                      h2.fix.intercept = F,\n",
    "                      LDSC = T,\n",
    "                      ldscore.dir = \"/import/home/share/xhu/database/1KG/eur_w_ld_chr\")\n",
    "    \n",
    "    write.table(data.frame(exposure,\n",
    "                           outcome,\n",
    "                           nrow(paras$dat),\n",
    "                           rg = paras$ldsc_res$rg,\n",
    "                           rg.se = paras$ldsc_res$rg.se,\n",
    "                           paras$ldsc_res$I[1,1],\n",
    "                           paras$ldsc_res$I[2,2],\n",
    "                           rho = paras$ldsc_res$I[1,2],\n",
    "                           paras$ldsc_res$I.se[1,1],\n",
    "                           paras$ldsc_res$I.se[2,2],\n",
    "                           rho = paras$ldsc_res$I.se[1,2],\n",
    "                           paras$ldsc_res$cov[1,1],  paras$ldsc_res$cov[1,2],  paras$ldsc_res$cov[2,2],\n",
    "                           paras$ldsc_res$cov.se[1,1],  paras$ldsc_res$cov.se[1,2],  paras$ldsc_res$cov.se[2,2]),\n",
    "                file = \"pairs_ldsc_res\", col.names = F, row.names = F, quote=F,append = T)\n",
    "    \n",
    "    if(inherits(paras, 'try-error')) next\n",
    "    \n",
    "    write.table(matrix(as.vector(paras$Omega), nrow=1), paste0(\"./pairs_bg_paras/\", exposure, \"~\", outcome, \"_Omega\"),\n",
    "                row.names = F, col.names = F, quote = F)\n",
    "      \n",
    "    write.table(matrix(as.vector(paras$C), nrow=1), paste0(\"./pairs_bg_paras/\", exposure, \"~\", outcome, \"_C\"),\n",
    "                row.names = F, col.names = F, quote = F)\n",
    "    \n",
    "    # clumping      \n",
    "    cat(\"Begin clumping ...\\n \")\n",
    "    \n",
    "    clumped =  clump(paras$dat,\n",
    "                     IV.Threshold = 5e-05,\n",
    "                     SNP_col = \"SNP\",\n",
    "                     pval_col = \"pval.exp\",\n",
    "                     clump_kb = 1000,\n",
    "                     clump_r2 = 0.001,\n",
    "                     plink = \"/import/home/maxhu/plink/plink\",\n",
    "                     bfile = \"/import/home/share/xhu/database/1KG/all_1000G_EUR_Phase3\")\n",
    "    \n",
    "    if(inherits(clumped , 'try-error')) next\n",
    "    \n",
    "    \n",
    "    write.table(clumped, file = paste0(\"./MRdat/\", exposure,\"~\",outcome), col.names=T, row.names=F, quote=F)\n",
    "    \n",
    "    # Stop the clock\n",
    "    proc.time() - start\n",
    "    \n",
    "  }\n",
    "  \n",
    "}\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
